diff -up VirtualBox-6.1.24/src/VBox/Additions/common/VBoxService/VBoxServiceAutoMount.cpp.9~ VirtualBox-6.1.24/src/VBox/Additions/common/VBoxService/VBoxServiceAutoMount.cpp
--- VirtualBox-6.1.24/src/VBox/Additions/common/VBoxService/VBoxServiceAutoMount.cpp.9~	2021-07-21 17:26:28.812790890 +0200
+++ VirtualBox-6.1.24/src/VBox/Additions/common/VBoxService/VBoxServiceAutoMount.cpp	2021-07-21 17:33:52.018092159 +0200
@@ -388,6 +388,13 @@ static int vbsvcAutoMountSharedFolderOld
                       0,                        /* int datalen */
                       szOptBuf,
                       sizeof(szOptBuf));
+        if (r == -1 && errno == EINVAL)
+        {
+            /* Mainline vboxsf accepts regular mount opts. */
+            char mount_opts[1024];
+            snprintf(mount_opts, 1024, "gid=%d,dmode=0770,fmode=0770", grp_vboxsf->gr_gid);
+            r = mount(pszShareName, pszMountPoint, "vboxsf", fFlags, mount_opts);
+        }
         if (r == 0)
             VGSvcVerbose(0, "vbsvcAutoMountWorker: Shared folder '%s' was mounted to '%s'\n", pszShareName, pszMountPoint);
         else if (errno != EBUSY) /* Share is already mounted? Then skip error msg. */
