diff -up VirtualBox-7.1.0_BETA1/src/VBox/Installer/linux/Makefile-header.gmk.14~ VirtualBox-7.1.0_BETA1/src/VBox/Installer/linux/Makefile-header.gmk
--- VirtualBox-7.1.0_BETA1/src/VBox/Installer/linux/Makefile-header.gmk.14~	2024-08-12 13:31:09.716391963 +0200
+++ VirtualBox-7.1.0_BETA1/src/VBox/Installer/linux/Makefile-header.gmk	2024-08-12 13:36:25.115834612 +0200
@@ -60,7 +60,7 @@
 #
 
 # VBOX_KBUILD_TARGET_ARCH = amd64|x86|arm64|arm32
-ifeq ($(filter-out x86_64 amd64 AMD64,$(shell uname -m)),)
+ifeq ($(filter-out x86_64 amd64 AMD64 znver1,$(shell rpm --eval %_target_cpu)),)
  VBOX_KBUILD_TARGET_ARCH_DEFAULT := amd64
 else
  ifeq ($(filter-out aarch64,$(shell uname -m)),)
