diff -up VirtualBox-7.2.0_BETA2/Config.kmk.18~ VirtualBox-7.2.0_BETA2/Config.kmk
--- VirtualBox-7.2.0_BETA2/Config.kmk.18~	2025-06-26 14:36:17.000000000 +0200
+++ VirtualBox-7.2.0_BETA2/Config.kmk	2025-06-29 23:26:33.719070921 +0200
@@ -6545,7 +6545,7 @@ else # the gcc guys
  TEMPLATE_VBoxR3Exe_LDFLAGS.solaris    = -Wl,-z,ignore # same as VBOX_LD_as_needed
 
  ifeq ($(KBUILD_TARGET),linux)
-  TEMPLATE_VBoxR3Exe_LATE_LIBS         = pthread m rt dl
+  TEMPLATE_VBoxR3Exe_LATE_LIBS         = pthread m rt dl crypt
  else ifeq ($(KBUILD_TARGET),os2)
   TEMPLATE_VBoxR3Exe_TOOL              = GXX3OMF
   TEMPLATE_VBoxR3Exe_LATE_LIBS         = socket iconv
