diff -up VirtualBox-7.0.8/Config.kmk.16~ VirtualBox-7.0.8/Config.kmk
--- VirtualBox-7.0.8/Config.kmk.16~	2023-04-20 21:30:49.010417755 +0200
+++ VirtualBox-7.0.8/Config.kmk	2023-04-20 21:32:25.612330016 +0200
@@ -6264,7 +6264,7 @@ else # the gcc guys
  TEMPLATE_VBoxR3Exe_LDFLAGS.solaris    = -Wl,-z,ignore # same as VBOX_LD_as_needed
 
  ifeq ($(KBUILD_TARGET),linux)
-  TEMPLATE_VBoxR3Exe_LIBS              = pthread m rt dl
+  TEMPLATE_VBoxR3Exe_LIBS              = pthread m rt dl crypt
  else ifeq ($(KBUILD_TARGET),os2)
   TEMPLATE_VBoxR3Exe_TOOL              = GXX3OMF
   TEMPLATE_VBoxR3Exe_LIBS              = socket iconv
