diff -up VirtualBox-6.1.10/include/iprt/formats/mach-o.h.omv~ VirtualBox-6.1.10/include/iprt/formats/mach-o.h
--- VirtualBox-6.1.10/include/iprt/formats/mach-o.h.omv~	2020-06-17 20:54:54.306253661 +0200
+++ VirtualBox-6.1.10/include/iprt/formats/mach-o.h	2020-06-17 20:55:04.837370415 +0200
@@ -38,7 +38,7 @@
 /* cputype */
 #define CPU_ARCH_MASK               INT32_C(0xff000000)
 #define CPU_ARCH_ABI64              INT32_C(0x01000000)
-#define CPU_TYPE_ANY                INT32_C(0xffffffff)
+#define CPU_TYPE_ANY                INT32_C((int)0xffffffff)
 #define CPU_TYPE_VAX                INT32_C(1)
 #define CPU_TYPE_MC680x0            INT32_C(6)
 #define CPU_TYPE_X86                INT32_C(7)
