diff -up VirtualBox-7.2.0_BETA2/src/VBox/Frontends/VirtualBox/src/runtime/UIMachineView.cpp.3~ VirtualBox-7.2.0_BETA2/src/VBox/Frontends/VirtualBox/src/runtime/UIMachineView.cpp
--- VirtualBox-7.2.0_BETA2/src/VBox/Frontends/VirtualBox/src/runtime/UIMachineView.cpp.3~	2025-06-26 14:49:53.000000000 +0200
+++ VirtualBox-7.2.0_BETA2/src/VBox/Frontends/VirtualBox/src/runtime/UIMachineView.cpp	2025-06-29 23:24:29.654689949 +0200
@@ -963,9 +963,9 @@ void UIMachineView::sltHandleActionTrigg
     {
         /* Defaults: */
         if (!uWidth)
-            uWidth = 800;
+            uWidth = 1024;
         if (!uHeight)
-            uHeight = 600;
+            uHeight = 768;
 
         /* Update current window size limitations: */
         setMaximumGuestSize(QSize((int)uWidth, (int)uHeight));
