diff -Naur VirtualBox-6.0.0/doc/manual/en_US/user_GuestAdditions.xml VirtualBox-6.0.0-mount/doc/manual/en_US/user_GuestAdditions.xml
--- VirtualBox-6.0.0/doc/manual/en_US/user_GuestAdditions.xml	2018-12-18 13:51:29.000000000 +0100
+++ VirtualBox-6.0.0-mount/doc/manual/en_US/user_GuestAdditions.xml	2018-12-29 18:10:23.293272432 +0100
@@ -1202,7 +1202,7 @@
             In a Linux guest, use the following command:
           </para>
 
-<screen>mount -t vboxsf [-o OPTIONS] sharename mountpoint</screen>
+<screen>mount -cit vboxsf [-o OPTIONS] sharename mountpoint</screen>
 
           <para>
             To mount a shared folder during boot, add the following
diff -Naur VirtualBox-6.0.0/doc/manual/fr_FR/user_GuestAdditions.xml VirtualBox-6.0.0-mount/doc/manual/fr_FR/user_GuestAdditions.xml
--- VirtualBox-6.0.0/doc/manual/fr_FR/user_GuestAdditions.xml	2018-12-18 13:51:29.000000000 +0100
+++ VirtualBox-6.0.0-mount/doc/manual/fr_FR/user_GuestAdditions.xml	2018-12-29 18:10:45.430391834 +0100
@@ -1167,7 +1167,7 @@
           <listitem>
             <para>Dans un invité Linux, utilisez la commande suivante&#xA0;:</para>
 
-            <screen>mount -t vboxsf [-o OPTIONS] sharename mountpoint</screen>
+            <screen>mount -cit vboxsf [-o OPTIONS] sharename mountpoint</screen>
 
             <para>Pour monter un dossier partagé au moment du démarrage, ajoutez
             l'entrée suivante à /etc/fstab&#xA0;:</para>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_bg.ts	2018-12-29 18:11:47.088724354 +0100
@@ -10023,8 +10023,8 @@
         <translation type="vanished">Само за четене</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Извежда всички споделени папки, достъпни за тази машина. Използвайте &apos;net use x: \\vboxsvr\share&apos;, за да достъпите споделена папка с име &lt;i&gt;share&lt;/i&gt; от DOS-подобни ОС, или &apos;mount -t vboxsf share mount_point&apos;, за да я достъпите от Linux ОС. Тази функция изисква Виртуални Добавки.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Извежда всички споделени папки, достъпни за тази машина. Използвайте &apos;net use x: \\vboxsvr\share&apos;, за да достъпите споделена папка с име &lt;i&gt;share&lt;/i&gt; от DOS-подобни ОС, или &apos;mount -cit vboxsf share mount_point&apos;, за да я достъпите от Linux ОС. Тази функция изисква Виртуални Добавки.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca.ts	2018-12-29 18:11:46.880723233 +0100
@@ -6266,8 +6266,8 @@
         <translation type="vanished">Només lectura</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Mostra totes les carpetes accessibles a aquesta màquina. Feu servir: &apos;net use x: \\vboxsvr\share&apos; per accedir a la carpeta anomenada &lt;i&gt;share&lt;/i&gt; des d&apos;un sistema tipus DOS, o &apos;mount -t vboxsf share mount_point&apos; per accedir a ella des d&apos;un sistema Linux. Aquesta característica requereix les Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Mostra totes les carpetes accessibles a aquesta màquina. Feu servir: &apos;net use x: \\vboxsvr\share&apos; per accedir a la carpeta anomenada &lt;i&gt;share&lt;/i&gt; des d&apos;un sistema tipus DOS, o &apos;mount -cit vboxsf share mount_point&apos; per accedir a ella des d&apos;un sistema Linux. Aquesta característica requereix les Guest Additions.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ca_VA.ts	2018-12-29 18:11:47.042724106 +0100
@@ -9875,8 +9875,8 @@
         <translation type="vanished">Només lectura</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Mostra totes les carpetes accessibles a esta màquina. Feu servir: &apos;net use x: \\vboxsvr\share&apos; per accedir a la carpeta anomenada &lt;i&gt;share&lt;/i&gt; des d&apos;un sistema tipus DOS, o &apos;mount -t vboxsf share mount_point&apos; per accedir a ella des d&apos;un sistema Linux. Esta característica requereix les Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Mostra totes les carpetes accessibles a esta màquina. Feu servir: &apos;net use x: \\vboxsvr\share&apos; per accedir a la carpeta anomenada &lt;i&gt;share&lt;/i&gt; des d&apos;un sistema tipus DOS, o &apos;mount -cit vboxsf share mount_point&apos; per accedir a ella des d&apos;un sistema Linux. Esta característica requereix les Guest Additions.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -22829,7 +22829,7 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Mostra totes les carpetes accessibles a esta màquina.
 Feu servir:
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_cs.ts	2018-12-29 18:11:46.785722720 +0100
@@ -10031,8 +10031,8 @@
         <translation type="vanished">Pouze pro čtení</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Zobrazí všechny sdílené složky dostupné tomuto počítači. Použijte příkaz &apos;net use x: \\vboxsvr\sdileni&apos; pro přístup ke sdílené složce s názvem &lt;i&gt;sdileni&lt;/i&gt; z OS DOSového typu nebo mount -t vboxsf share pripojny_bod pro přístup z Linuxových OS. Tato funkce vyžaduje Přídavky pro hosta.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Zobrazí všechny sdílené složky dostupné tomuto počítači. Použijte příkaz &apos;net use x: \\vboxsvr\sdileni&apos; pro přístup ke sdílené složce s názvem &lt;i&gt;sdileni&lt;/i&gt; z OS DOSového typu nebo mount -cit vboxsf share pripojny_bod pro přístup z Linuxových OS. Tato funkce vyžaduje Přídavky pro hosta.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -22743,13 +22743,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Zobrazí všechny sdílené adresáře dostupné tomuto počítači
 Použijte
 &lt;tt&gt;net use x: \\vboxsvr\sdileni&lt;/tt&gt;
 pro přístup ke sdílenému adresáři jménem &lt;i&gt;sdileni&lt;/i&gt; z OS DOSového typu nebo
-&lt;tt&gt;mount -t vboxsf share pripojny_bod&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share pripojny_bod&lt;/tt&gt;
 pro přístup z Linuxových OS. Tato funkce vyžaduje Přídavky pro hosta.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_da.ts	2018-12-29 18:11:46.964723686 +0100
@@ -6237,8 +6237,8 @@
         <translation type="vanished">Skrivebeskyttet</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Viser alle delte mapper tilgængelige for denne maskine. Brug &apos;net use x: \\vboxsvr\share&apos; på Windows eller &apos;mount -t vboxsf share mount_point&apos; på Linux for at tilgå den delte mappe &lt;i&gt;share&lt;/i&gt;. Denne funktion kræver at gæstetilføjelser er installeret.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Viser alle delte mapper tilgængelige for denne maskine. Brug &apos;net use x: \\vboxsvr\share&apos; på Windows eller &apos;mount -cit vboxsf share mount_point&apos; på Linux for at tilgå den delte mappe &lt;i&gt;share&lt;/i&gt;. Denne funktion kræver at gæstetilføjelser er installeret.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_de.ts	2018-12-29 18:11:46.897723324 +0100
@@ -6831,8 +6831,8 @@
         <translation type="vanished">Nur lesbar</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Zeigt alle von dieser virtuellen Maschine zugreifbaren gemeinsamen Ordner mit dem Host. Benutzen Sie &apos;net use x: \\vboxsvr\share&apos; um auf einen gemeinsamen Ordner namens &lt;i&gt;share&lt;/i&gt; von einem DOS-artigen BS bzw. &apos;mount -t vboxsf share mount_point&apos; um von Linux darauf zuzugreifen. Dieses Merkmal benötigt Gasterweiterungen.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Zeigt alle von dieser virtuellen Maschine zugreifbaren gemeinsamen Ordner mit dem Host. Benutzen Sie &apos;net use x: \\vboxsvr\share&apos; um auf einen gemeinsamen Ordner namens &lt;i&gt;share&lt;/i&gt; von einem DOS-artigen BS bzw. &apos;mount -cit vboxsf share mount_point&apos; um von Linux darauf zuzugreifen. Dieses Merkmal benötigt Gasterweiterungen.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_el.ts	2018-12-29 18:11:47.114724494 +0100
@@ -6881,8 +6881,8 @@
         <translation type="vanished">Μόνο ανάγνωση</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Δείχνει όλους τους κοινόχρηστους φακέλους προσβάσιμους απ&apos; αυτή τη μηχανή. Χρησιμοποιήστε &apos;net use x: \\vboxsvr\share&apos; για πρόσβαση του κοινόχρηστου φακέλου ονόματι &lt;i&gt;share&lt;/i&gt; από ένα λειτουργικό βασισμένο στο DOS, ή &apos;mount -t vboxsf share mount_point&apos; για πρόσβαση του κοινόχρηστου φακέλου από ένα λειτουργικό βασισμένο στο Linux. Αυτό το χαρακτηριστικό απαιτεί τις Προσθήκες Επισκέπτη.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Δείχνει όλους τους κοινόχρηστους φακέλους προσβάσιμους απ&apos; αυτή τη μηχανή. Χρησιμοποιήστε &apos;net use x: \\vboxsvr\share&apos; για πρόσβαση του κοινόχρηστου φακέλου ονόματι &lt;i&gt;share&lt;/i&gt; από ένα λειτουργικό βασισμένο στο DOS, ή &apos;mount -cit vboxsf share mount_point&apos; για πρόσβαση του κοινόχρηστου φακέλου από ένα λειτουργικό βασισμένο στο Linux. Αυτό το χαρακτηριστικό απαιτεί τις Προσθήκες Επισκέπτη.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_es.ts	2018-12-29 18:11:46.889723281 +0100
@@ -10385,8 +10385,8 @@
         <translation type="vanished">Sólo lectura</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Lista todas las carpetas compartidas accesibles por esta máquina. Use «net use x: \\vboxsvr\share» para acceder a una carpeta compartida llamada &lt;i&gt;share&lt;/i&gt; desde un SO similar a DOS o «mount -t vboxsf share mount_point» para acceder desde un SO Linux. Esta característica requiere las «Guest Additions».</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Lista todas las carpetas compartidas accesibles por esta máquina. Use «net use x: \\vboxsvr\share» para acceder a una carpeta compartida llamada &lt;i&gt;share&lt;/i&gt; desde un SO similar a DOS o «mount -cit vboxsf share mount_point» para acceder desde un SO Linux. Esta característica requiere las «Guest Additions».</translation>
     </message>
     <message>
         <source>Name</source>
@@ -24239,12 +24239,12 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Despliega una lista de todas las carpetas compartidas a esta máquina.
 Utilice &lt;tt&gt;net use x: \\vboxsvr\prestamo&lt;/tt&gt;
 para acceder a la carpeta compartido &lt;i&gt;prestamo&lt;/i&gt; desde un OS de tipo windows.
-O &lt;tt&gt;mount -t vboxsf prestamo mount_point&lt;/tt&gt;
+O &lt;tt&gt;mount -cit vboxsf prestamo mount_point&lt;/tt&gt;
 para acceder desde un OS Linux.
 Esta funcionalidad requiere &quot;Guest Additions&quot; instalado.&lt;/qt&gt;</translation>
     </message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_eu.ts	2018-12-29 18:11:47.094724387 +0100
@@ -9464,8 +9464,8 @@
     </message>
     <message>
         <location line="+37"/>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Makina honetatik eskuragarri dauden elkarbanatutako agiritegi guztiak zerrendatzen ditu. Erabili  &apos;net use x: \\vboxsvr\share&apos; &lt;i&gt;share&lt;/i&gt; izeneko elkarbanatutako agiritegi batera sarbidea izateko, edo  &apos;mount -t vboxsf share mount_point&apos; Linux SE batetik sarbidea izateko. Ezaugarri honek Gonbidatu Gehigarriak behar ditu.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Makina honetatik eskuragarri dauden elkarbanatutako agiritegi guztiak zerrendatzen ditu. Erabili  &apos;net use x: \\vboxsvr\share&apos; &lt;i&gt;share&lt;/i&gt; izeneko elkarbanatutako agiritegi batera sarbidea izateko, edo  &apos;mount -cit vboxsf share mount_point&apos; Linux SE batetik sarbidea izateko. Ezaugarri honek Gonbidatu Gehigarriak behar ditu.</translation>
     </message>
     <message>
         <location line="+10"/>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fa.ts	2018-12-29 18:11:46.800722801 +0100
@@ -8247,8 +8247,8 @@
         <translation type="vanished">فقط-خواندنی</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>لیست همه پوشه های قابل دسترسی به این ماشین. برای دسترسی به پوشه اشتراکی از &apos;net use x: \\vboxsvr\share&apos; استفاده کنید که نام دارد &lt;i&gt;share&lt;/i&gt; از سیستم عامل شبیه داس, یا &apos;mount -t vboxsf share mount_point&apos; برای دسترسی آن از سیستم عامل لینوکس. این ویژگی نیاز به افزونه مهمان دارد.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>لیست همه پوشه های قابل دسترسی به این ماشین. برای دسترسی به پوشه اشتراکی از &apos;net use x: \\vboxsvr\share&apos; استفاده کنید که نام دارد &lt;i&gt;share&lt;/i&gt; از سیستم عامل شبیه داس, یا &apos;mount -cit vboxsf share mount_point&apos; برای دسترسی آن از سیستم عامل لینوکس. این ویژگی نیاز به افزونه مهمان دارد.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fi.ts	2018-12-29 18:11:46.777722677 +0100
@@ -8137,8 +8137,8 @@
         <translation type="vanished">Vain luku</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Näyttää luettelon kaikista käytettävissä olevista jaetuista kansioista. Pääset käsiksi jaettuun &lt;i&gt;levyjakoon&lt;/i&gt; DOS-tyylisistä virtuaalikoneista komennolla &apos;net use x: \\vboxsvr\share&apos; ja linux-virtuaalikoneista komennolla &apos;mount -t vboxsf levyjako liitospiste&apos;. Tämä ominaisuus vaatii asiakaslisäosien asennuksen. </translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Näyttää luettelon kaikista käytettävissä olevista jaetuista kansioista. Pääset käsiksi jaettuun &lt;i&gt;levyjakoon&lt;/i&gt; DOS-tyylisistä virtuaalikoneista komennolla &apos;net use x: \\vboxsvr\share&apos; ja linux-virtuaalikoneista komennolla &apos;mount -cit vboxsf levyjako liitospiste&apos;. Tämä ominaisuus vaatii asiakaslisäosien asennuksen. </translation>
     </message>
     <message>
         <source>Name</source>
@@ -18340,13 +18340,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Luettelee kaikki jaetut kansiot jotka ovat käyttävissä tällä koneella.
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_fr.ts	2018-12-29 18:11:47.030724041 +0100
@@ -9370,8 +9370,8 @@
         <translation type="vanished">Lecture seule</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Affiche tous les dossiers partagés accessibles par cette machine. Exécutez « net use x: \\vboxsvr\&lt;dossier&gt; » pour accéder à &lt;dossier&gt; depuis un système d&apos;exploitation DOS, ou bien « mount -t vboxsf &lt;dossier&gt; mount_point »  pour y accéder depuis Linux. Cette fonctionnalité nécessite l&apos;installation des Additions invité.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Affiche tous les dossiers partagés accessibles par cette machine. Exécutez « net use x: \\vboxsvr\&lt;dossier&gt; » pour accéder à &lt;dossier&gt; depuis un système d&apos;exploitation DOS, ou bien « mount -cit vboxsf &lt;dossier&gt; mount_point »  pour y accéder depuis Linux. Cette fonctionnalité nécessite l&apos;installation des Additions invité.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_gl.ts	2018-12-29 18:11:46.939723551 +0100
@@ -8053,8 +8053,8 @@
         <translation type="vanished">Só de lectura</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Lista todos os cartafoles compartidos accesíbeis a esta máquina. Empregue &apos;net use x: \\vboxsvr\\share&apos; para acceder a un cartafol chamado &lt;i&gt;compartir&lt;/i&gt; desde un sistema operativo baseado en DOS, ou &apos;mount -t vboxsf compartir punto_de_montaxe&apos; para acceder desde un sistema operativo Linux. Esta funcionalidade require dos aplicativos para o sistema operativo convidado.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Lista todos os cartafoles compartidos accesíbeis a esta máquina. Empregue &apos;net use x: \\vboxsvr\\share&apos; para acceder a un cartafol chamado &lt;i&gt;compartir&lt;/i&gt; desde un sistema operativo baseado en DOS, ou &apos;mount -cit vboxsf compartir punto_de_montaxe&apos; para acceder desde un sistema operativo Linux. Esta funcionalidade require dos aplicativos para o sistema operativo convidado.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_he.ts	2018-12-29 18:11:46.943723572 +0100
@@ -6315,7 +6315,7 @@
         <translation type="vanished">קריאה-בלבד</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hr_HR.ts	2018-12-29 18:11:46.996723858 +0100
@@ -6352,8 +6352,8 @@
         <translation type="vanished">Samo za čitanje</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Popisuje sve dijeljene mape dostupne ovoj mašini. Koristite &apos;net use x: \\vboxsvr\share&apos; da biste dobili pristup dijeljenoj mapi naziva &lt;i&gt;share&lt;/i&gt; iz OS-a sličnom DOS-u ili &apos;mount -t vboxsf share mount_point&apos; da biste dobili pristup iz Linux OS-a. Ova značajka zahtijeva Gostne Dodatke.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Popisuje sve dijeljene mape dostupne ovoj mašini. Koristite &apos;net use x: \\vboxsvr\share&apos; da biste dobili pristup dijeljenoj mapi naziva &lt;i&gt;share&lt;/i&gt; iz OS-a sličnom DOS-u ili &apos;mount -cit vboxsf share mount_point&apos; da biste dobili pristup iz Linux OS-a. Ova značajka zahtijeva Gostne Dodatke.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_hu.ts	2018-12-29 18:11:46.919723443 +0100
@@ -10081,8 +10081,8 @@
         <translation type="vanished">Csak olvasható</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Minden a gép számára elérhető megosztott mappa listázása. A &apos;net use x: \\vboxsvr\share&apos; parancs segítségével használható a &lt;i&gt;share&lt;/i&gt; nevű megosztás a DOS-szerű OS-ekből, a &apos;mount -t vboxsf share mount_point&apos; paranccsal pedig Linux-alapú OS-ekből. Az integrációs szolgáltatások telepítése szükséges hozzá.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Minden a gép számára elérhető megosztott mappa listázása. A &apos;net use x: \\vboxsvr\share&apos; parancs segítségével használható a &lt;i&gt;share&lt;/i&gt; nevű megosztás a DOS-szerű OS-ekből, a &apos;mount -cit vboxsf share mount_point&apos; paranccsal pedig Linux-alapú OS-ekből. Az integrációs szolgáltatások telepítése szükséges hozzá.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -22846,13 +22846,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Minden a gép számára elérhető megosztott mappa listázása.
 Használja a
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 parancsot a &lt;i&gt;share&lt;/i&gt; nevű megosztás eléréséhez a DOS-szerű OS-ekből, vagy
-a &lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+a &lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 parancsot Linux-alapú OS-ekből. Integrációs szolgáltatások szükségesek hozzá.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_id.ts	2018-12-29 18:11:46.959723659 +0100
@@ -9604,8 +9604,8 @@
         <translation type="vanished">Baca saja</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Menampilkan daftar semua folder bersama yang dapat diakses oleh mesin ini. Gunakan &apos;net use x: \\vboxsvr\share&apos; untuk mengakses sebuah folder bersama bernama &lt;i&gt;share&lt;/i&gt; dari OS serupa DOS atau &apos;mount -t vboxsf share titik_kait&apos; dari OS Linux. Fitur ini membutuhkan Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Menampilkan daftar semua folder bersama yang dapat diakses oleh mesin ini. Gunakan &apos;net use x: \\vboxsvr\share&apos; untuk mengakses sebuah folder bersama bernama &lt;i&gt;share&lt;/i&gt; dari OS serupa DOS atau &apos;mount -cit vboxsf share titik_kait&apos; dari OS Linux. Fitur ini membutuhkan Guest Additions.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -21947,13 +21947,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Menampilkan daftar semua folder yuang di share yang dapat diakses melalui mesin ini.
 Menggunakan
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 untuk mengakses sebuah folder yang di share dengan nama &lt;i&gt;share&lt;/i&gt; dari Sistem Operasi serupa DOS atau
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;(new line)
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;(new line)
 untuk mengaksesnya dari sebuah Sistem Operasi Linux. Fitur ini membutuhkan Guest Additions.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_it.ts	2018-12-29 18:11:47.104724441 +0100
@@ -11248,8 +11248,8 @@
         <translation type="vanished">Sola lettura</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Elenca tutte le cartelle condivise accessibili a questa macchina. Utilizza &apos;net use x: \\vboxsvr\share&apos; per accedere alla cartella condivisa con nome &lt;i&gt;share&lt;/i&gt; da un sistema operativo tipo DOS, o &apos;mount -t vboxsf share mount_point&apos; per accedervi da un sistema Linux. Questa funzionalità richiede le Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Elenca tutte le cartelle condivise accessibili a questa macchina. Utilizza &apos;net use x: \\vboxsvr\share&apos; per accedere alla cartella condivisa con nome &lt;i&gt;share&lt;/i&gt; da un sistema operativo tipo DOS, o &apos;mount -cit vboxsf share mount_point&apos; per accedervi da un sistema Linux. Questa funzionalità richiede le Guest Additions.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -24982,13 +24982,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Elenca tutte le cartelle condivise accessibili da questa macchina.
 Usa
 &lt;tt&gt;net use x: \\vboxsrv\share&lt;/tt&gt;
 per accedere ad una cartella condivisa chiamata &lt;i&gt;share&lt;/i&gt; da un sistema basato su DOS, oppure
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 per accedervi da un sistema Linux. Questa funzionalità richiede Guest Additions.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ja.ts	2018-12-29 18:11:46.863723141 +0100
@@ -10414,8 +10414,8 @@
         <translation type="vanished">読み込み専用</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>この仮想マシンでアクセス可能なすべての共有フォルダーをリスト表示します。DOS/Windows系ゲストOSからアクセスする場合は &lt;tt&gt;net use x:¥¥vboxsvr¥share&lt;/tt&gt; コマンドを使用してください。&lt;i&gt;share&lt;/i&gt;はアクセスしたいフォルダーの名前です。 LinuxゲストOSからアクセスする場合は &lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt; コマンドを使用してください。本機能を使用するには Guest Additions のインストールが必要です。</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>この仮想マシンでアクセス可能なすべての共有フォルダーをリスト表示します。DOS/Windows系ゲストOSからアクセスする場合は &lt;tt&gt;net use x:¥¥vboxsvr¥share&lt;/tt&gt; コマンドを使用してください。&lt;i&gt;share&lt;/i&gt;はアクセスしたいフォルダーの名前です。 LinuxゲストOSからアクセスする場合は &lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt; コマンドを使用してください。本機能を使用するには Guest Additions のインストールが必要です。</translation>
     </message>
     <message>
         <source>Name</source>
@@ -23478,11 +23478,11 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;この仮想マシンでアクセス可能なすべての共有フォルダーのリストです。
 DOS系ゲストOSからアクセスする場合は &lt;tt&gt;net use x:¥¥vboxsvr¥share&lt;/tt&gt; コマンドを使用してください。
-&lt;i&gt;share&lt;/i&gt;はアクセスしたいフォルダーの名前です。 LinuxゲストOSからアクセスする場合は &lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt; コマンドを使用してください。
+&lt;i&gt;share&lt;/i&gt;はアクセスしたいフォルダーの名前です。 LinuxゲストOSからアクセスする場合は &lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt; コマンドを使用してください。
 本機能を使用するには Guest Additions のインストールが必要です。&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_km_KH.ts	2018-12-29 18:11:47.002723891 +0100
@@ -7780,8 +7780,8 @@
         <translation type="vanished">បាន​តែ​អាន</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>រាយ​ថត​ដែល​បានចែករំលែក​ទាំង​អស់​ដែល​អាច​ចូលដំណើរការ​បានទៅ​ម៉ាស៊ីន​នេះ&#xa0;។ ប្រើ​ &apos;net ប្រើ x: \\vboxsvr\share&apos; ដើម្បី​ចូលដំណើរការ​ថត​ដែល​បានចែករំលែក​ដែលមាន​ឈ្មោះ &lt;i&gt;ចែករំលែក&lt;/i&gt; ពីប្រព័ន្ធ​ប្រតិបត្តិការ​ដូចនឹង DOS ដែរ ឬ​ &apos;mount -t vboxsf share mount_point&apos; ដើម្បីចូលដំណើរការ​វា​ពី​ប្រព័ន្ធ​ប្រតិបត្តិការ​លីនុច&#xa0;។ លក្ខណៈពិសេស​នេះ​ត្រូវការ​ផ្នែក​បន្ថែម​របស់ម៉ាស៊ីន​ភ្ញៀវ&#xa0;។</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>រាយ​ថត​ដែល​បានចែករំលែក​ទាំង​អស់​ដែល​អាច​ចូលដំណើរការ​បានទៅ​ម៉ាស៊ីន​នេះ&#xa0;។ ប្រើ​ &apos;net ប្រើ x: \\vboxsvr\share&apos; ដើម្បី​ចូលដំណើរការ​ថត​ដែល​បានចែករំលែក​ដែលមាន​ឈ្មោះ &lt;i&gt;ចែករំលែក&lt;/i&gt; ពីប្រព័ន្ធ​ប្រតិបត្តិការ​ដូចនឹង DOS ដែរ ឬ​ &apos;mount -cit vboxsf share mount_point&apos; ដើម្បីចូលដំណើរការ​វា​ពី​ប្រព័ន្ធ​ប្រតិបត្តិការ​លីនុច&#xa0;។ លក្ខណៈពិសេស​នេះ​ត្រូវការ​ផ្នែក​បន្ថែម​របស់ម៉ាស៊ីន​ភ្ញៀវ&#xa0;។</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ko.ts	2018-12-29 18:11:47.023724004 +0100
@@ -10407,8 +10407,8 @@
         <translation type="vanished">읽기 전용</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>이 가상 머신에서 사용 가능한 공유 폴더의 목록입니다. 도스 계열 운영 체제에서 &lt;i&gt;share&lt;/i&gt;라는 공유 폴더에 접근하려면 &apos;net use x: \\vboxsvr\share&apos;와 같은 식으로 입력하시고, 리눅스에서는 &apos;mount -t vboxsf share mount_point&apos;와 같이 입력하십시오. 이 기능을 사용하려면 게스트 확장이 필요합니다.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>이 가상 머신에서 사용 가능한 공유 폴더의 목록입니다. 도스 계열 운영 체제에서 &lt;i&gt;share&lt;/i&gt;라는 공유 폴더에 접근하려면 &apos;net use x: \\vboxsvr\share&apos;와 같은 식으로 입력하시고, 리눅스에서는 &apos;mount -cit vboxsf share mount_point&apos;와 같이 입력하십시오. 이 기능을 사용하려면 게스트 확장이 필요합니다.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_lt.ts	2018-12-29 18:11:46.971723723 +0100
@@ -9065,8 +9065,8 @@
         <translation type="vanished">&amp;Aplankų sąrašas</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Pateikia visų šiai mašinai pasiekiamų bendrųjų aplankų sąrašą. Norėdami pasiekti aplanką &lt;i&gt;bendrasis&lt;/i&gt; iš DOS tipo OS, pvz., Windows, įvykdykite „net use x: \\vboxsvr\bendrasis“, o norėdami pasiekti iš Linux OS – įvykdykite „mount -t vboxsf bendrasis /prijungimo/vieta“. Ši funkcija galima tik įdiegus „svečio papildinius“.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Pateikia visų šiai mašinai pasiekiamų bendrųjų aplankų sąrašą. Norėdami pasiekti aplanką &lt;i&gt;bendrasis&lt;/i&gt; iš DOS tipo OS, pvz., Windows, įvykdykite „net use x: \\vboxsvr\bendrasis“, o norėdami pasiekti iš Linux OS – įvykdykite „mount -cit vboxsf bendrasis /prijungimo/vieta“. Ši funkcija galima tik įdiegus „svečio papildinius“.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -21617,8 +21617,8 @@
         <translation type="obsolete">Tik skaityti</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation type="obsolete">Pateikia visų šiai mašinai pasiekiamų bendrųjų aplankų sąrašą. Norėdami pasiekti aplanką &lt;i&gt;bendrasis&lt;/i&gt; iš DOS tipo OS, pvz., Windows, įvykdykite „net use x: \\vboxsvr\bendrasis“, o norėdami pasiekti iš Linux OS – įvykdykite „mount -t vboxsf bendrasis /prijungimo/vieta“. Ši funkcija galima tik įdiegus „svečio papildinius“.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation type="obsolete">Pateikia visų šiai mašinai pasiekiamų bendrųjų aplankų sąrašą. Norėdami pasiekti aplanką &lt;i&gt;bendrasis&lt;/i&gt; iš DOS tipo OS, pvz., Windows, įvykdykite „net use x: \\vboxsvr\bendrasis“, o norėdami pasiekti iš Linux OS – įvykdykite „mount -cit vboxsf bendrasis /prijungimo/vieta“. Ši funkcija galima tik įdiegus „svečio papildinius“.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_nl.ts	2018-12-29 18:11:46.875723206 +0100
@@ -6850,8 +6850,8 @@
         <translation type="vanished">Alleen-lezen</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Toont een lijst met alle gedeelde mappen die toegankelijk zijn voor deze machine. Gebruik &apos;net use x: \\vboxsvr\share&apos; om de gedeelde map genaamd &lt;i&gt;share&lt;/i&gt; te benaderen in een op MS-DOS gebaseert besturingssysteem (zoals Windows). Gebruik &apos;mount -t vboxsf share mount_point&apos; om deze gedeelde map in een Linux-omgeving te benaderen. Deze optie vereist de Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Toont een lijst met alle gedeelde mappen die toegankelijk zijn voor deze machine. Gebruik &apos;net use x: \\vboxsvr\share&apos; om de gedeelde map genaamd &lt;i&gt;share&lt;/i&gt; te benaderen in een op MS-DOS gebaseert besturingssysteem (zoals Windows). Gebruik &apos;mount -cit vboxsf share mount_point&apos; om deze gedeelde map in een Linux-omgeving te benaderen. Deze optie vereist de Guest Additions.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pl.ts	2018-12-29 18:11:46.908723384 +0100
@@ -9280,8 +9280,8 @@
         <translation type="vanished">Tylko do odczytu</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>&lt;qt&gt;Przechowuje listę udostępnianych folderów dostępnych dla tej maszyny. Użycie &apos;net use x: \\vboxsvr\udział&apos; pozwala uzyskać dostęp do współdzielonego folderu o nazwie &lt;i&gt;udział&lt;/i&gt; z systemu typu DOS, podobnie &apos;mount -t vboxsf udział punkt_montowania&apos; pozwala uzyskać dostęp do niego z poziomu systemu Linux. Uwaga: funkcja ta wymaga zainstalowania Dodatków gościa.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>&lt;qt&gt;Przechowuje listę udostępnianych folderów dostępnych dla tej maszyny. Użycie &apos;net use x: \\vboxsvr\udział&apos; pozwala uzyskać dostęp do współdzielonego folderu o nazwie &lt;i&gt;udział&lt;/i&gt; z systemu typu DOS, podobnie &apos;mount -cit vboxsf udział punkt_montowania&apos; pozwala uzyskać dostęp do niego z poziomu systemu Linux. Uwaga: funkcja ta wymaga zainstalowania Dodatków gościa.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -21351,14 +21351,14 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Przechowuje listę współdzielonych folderów dostępnych dla tej maszyny.
 Użycie
 &lt;tt&gt;net use x: \\vboxsvr\udział&lt;/tt&gt;
 pozwala uzyskać dostęp do współdzielonego folderu o nazwie &lt;i&gt;udział&lt;/i&gt;
 z systemu typu DOS, podobnie
-&lt;tt&gt;mount -t vboxsf udział punkt_montowania&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf udział punkt_montowania&lt;/tt&gt;
 pozwala uzyskać dostęp do niego z poziomu systemu Linux.
 Uwaga: funkcja ta wymaga zainstalowania Dodatków gościa.&lt;/qt&gt;</translation>
     </message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt_BR.ts	2018-12-29 18:11:47.015723961 +0100
@@ -10827,8 +10827,8 @@
         <translation type="vanished">Apenas para Leitura</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Lista todas as pastas compartilhadas acessíveis nesta máquina. Utilize &apos;net use x: \\vboxsvr\compartilhamento&apos; para acessar uma pasta compartilhada chamada &lt;i&gt;compartilhamento&lt;/i&gt; de um sistema operacional baseado em  DOS, ou &apos;mount -t vboxsf compartilhamento ponto_de_montagem&apos; para acessá-lo de um sistema operacional Linux. Este recurso requer a instalação dos Adicionais Para Convidado.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Lista todas as pastas compartilhadas acessíveis nesta máquina. Utilize &apos;net use x: \\vboxsvr\compartilhamento&apos; para acessar uma pasta compartilhada chamada &lt;i&gt;compartilhamento&lt;/i&gt; de um sistema operacional baseado em  DOS, ou &apos;mount -cit vboxsf compartilhamento ponto_de_montagem&apos; para acessá-lo de um sistema operacional Linux. Este recurso requer a instalação dos Adicionais Para Convidado.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -24669,13 +24669,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Lista todas as pastas compartilhadas acessíveis nesta máquina.
 Utilize
 &lt;tt&gt;net use x: \\vboxsvr\compartilhamento&lt;/tt&gt;
 para acessar uma pasta compartilhada chamada &lt;i&gt;compartilhamento&lt;/i&gt; de um sistema operacional baseado em  DOS, ou
-&lt;tt&gt;mount -t vboxsf compartilhamento ponto_de_montagem&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf compartilhamento ponto_de_montagem&lt;/tt&gt;
 para acessá-lo de um sistema operacional Linux. Este recurso requer a instalação dos Adicionais Para Convidado.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_pt.ts	2018-12-29 18:11:46.834722985 +0100
@@ -8481,8 +8481,8 @@
         <translation type="vanished">Leitura apenas</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Lista todas as pastas partilhadas nesta máquina. Use &apos;net use x: \\vboxsvr\partilha&apos; para aceder à pasta partilhada chamada &lt;i&gt;partilha&lt;/i&gt; num SO do tipo DOS, ou &apos;mount -t vboxsf partilha ponto_de_montagem&apos; para aceder à pasta num SO Linux. Esta opção requer as Adições de Convidado.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Lista todas as pastas partilhadas nesta máquina. Use &apos;net use x: \\vboxsvr\partilha&apos; para aceder à pasta partilhada chamada &lt;i&gt;partilha&lt;/i&gt; num SO do tipo DOS, ou &apos;mount -cit vboxsf partilha ponto_de_montagem&apos; para aceder à pasta num SO Linux. Esta opção requer as Adições de Convidado.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -19760,13 +19760,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Lista todas as pastas partilhadas acessíveis a esta máquina.
 Use
 &lt;tt&gt;net use x: \\vboxsvr\partilha&lt;/tt&gt;
 para aceder a uma pasta partilhada chamada &lt;i&gt;partilha&lt;/i&gt; de um SO tipo DOS,
-ou (a partir de um terminal) &lt;tt&gt;mount -t vboxsf partilha ponto_de_montagem&lt;/tt&gt;
+ou (a partir de um terminal) &lt;tt&gt;mount -cit vboxsf partilha ponto_de_montagem&lt;/tt&gt;
 para a aceder de um SO Linux. Este recurso requer a instalação das Adições de Convidado.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ro.ts	2018-12-29 18:11:47.080724311 +0100
@@ -8081,8 +8081,8 @@
         <translation type="vanished">Doar citire</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Listează toate directoarele partajate accesibile acestei mașini.Folosiți comanda:&lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;pentru a accesa un director partajat denumit &lt;i&gt;share&lt;/i&gt; dintr-un SO gen DOS, sau&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;pentru a-l accesa dintr-un SO Linux. Această funcție necesită Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Listează toate directoarele partajate accesibile acestei mașini.Folosiți comanda:&lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;pentru a accesa un director partajat denumit &lt;i&gt;share&lt;/i&gt; dintr-un SO gen DOS, sau&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;pentru a-l accesa dintr-un SO Linux. Această funcție necesită Guest Additions.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -19475,13 +19475,13 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Listează toate directoarele partajate accesibile acestei mașini.
 Folosiți comanda:
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 pentru a accesa un director partajat denumit &lt;i&gt;share&lt;/i&gt; dintr-un SO gen DOS, sau
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 pentru a-l accesa dintr-un SO Linux. Această funcție necesită Guest Additions.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_ru.ts	2018-12-29 18:11:47.064724225 +0100
@@ -10336,8 +10336,8 @@
         <translation type="vanished">Чтение</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Содержит список всех общих папок, доступных этой машине. Используйте команду &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt; для доступа к общей папке с именем &lt;i&gt;share&lt;/i&gt; в DOS-подобной ОС или &lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt; для доступа из Линукс-подобной ОС. Требует установки Дополнений гостевой ОС.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Содержит список всех общих папок, доступных этой машине. Используйте команду &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt; для доступа к общей папке с именем &lt;i&gt;share&lt;/i&gt; в DOS-подобной ОС или &lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt; для доступа из Линукс-подобной ОС. Требует установки Дополнений гостевой ОС.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sk.ts	2018-12-29 18:11:47.051724155 +0100
@@ -8476,8 +8476,8 @@
         <translation type="vanished">Iba na čítanie</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Zobrazí všetky zdieľané priečinky dostupné pre tento počítač. Použite &apos;net use x: \\vboxsrv\share&apos; pre prístup k zdieľanému priečinku s názvom &lt;i&gt;share&lt;/i&gt; pre operačný systém na báze DOSu alebo &apos;mount -t vboxsf share mount_point&apos; pre prístup k priečinku z operačného systému Linux. Táto vlastnosť vyžaduje inštaláciu Hosťovských doplnkov.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Zobrazí všetky zdieľané priečinky dostupné pre tento počítač. Použite &apos;net use x: \\vboxsrv\share&apos; pre prístup k zdieľanému priečinku s názvom &lt;i&gt;share&lt;/i&gt; pre operačný systém na báze DOSu alebo &apos;mount -cit vboxsf share mount_point&apos; pre prístup k priečinku z operačného systému Linux. Táto vlastnosť vyžaduje inštaláciu Hosťovských doplnkov.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sl.ts	2018-12-29 18:11:46.806722833 +0100
@@ -8280,8 +8280,8 @@
     </message>
     <message>
         <location line="+37"/>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Navede vse deljene mape, ki so dostopne temu računalniku. Za dostop do deljene mape z imenom &lt;i&gt;share&lt;/i&gt; uporabite &apos;net use x: \\vboxsvr\share&apos; iz OS-a vrste DOS ali &apos;mount -t vboxsf share mount_point&apos; iz OS-a Linux. Ta značilnost zahteva program Guest Additions.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Navede vse deljene mape, ki so dostopne temu računalniku. Za dostop do deljene mape z imenom &lt;i&gt;share&lt;/i&gt; uporabite &apos;net use x: \\vboxsvr\share&apos; iz OS-a vrste DOS ali &apos;mount -cit vboxsf share mount_point&apos; iz OS-a Linux. Ta značilnost zahteva program Guest Additions.</translation>
     </message>
     <message>
         <location line="+10"/>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sr.ts	2018-12-29 18:11:47.121724532 +0100
@@ -8832,8 +8832,8 @@
         <translation type="vanished">Само читљиво</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Списак дељених фасцикла на ову машину. Користити „net use x: \\vboxsvr\share“ за приступ дељену фасциклу именовану &lt;i&gt;share&lt;/i&gt; за ОС као DOS, или „mount -t vboxsf share mount_point“за приступ са линукса. За ово су потребни &apos;додатци госта&apos;.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Списак дељених фасцикла на ову машину. Користити „net use x: \\vboxsvr\share“ за приступ дељену фасциклу именовану &lt;i&gt;share&lt;/i&gt; за ОС као DOS, или „mount -cit vboxsf share mount_point“за приступ са линукса. За ово су потребни &apos;додатци госта&apos;.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_sv.ts	2018-12-29 18:11:46.927723486 +0100
@@ -9949,8 +9949,8 @@
         <translation type="vanished">Skrivskyddad</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Listar alla delade mappar åtkomliga för denna maskin. Använd \&quot;net use x: \\vboxsvr\utdelning\&quot; för att komma åt en delad mapp med namnet &lt;i&gt;utdelning&lt;/i&gt; från ett DOS-liknande operativsystem, eller \&quot;mount -t vboxsf utdelning monteringspunkt\&quot; för att komma åt den från ett operativsystem baserat på Linux. Denna funktion kräver Gästtillägg.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Listar alla delade mappar åtkomliga för denna maskin. Använd \&quot;net use x: \\vboxsvr\utdelning\&quot; för att komma åt en delad mapp med namnet &lt;i&gt;utdelning&lt;/i&gt; från ett DOS-liknande operativsystem, eller \&quot;mount -cit vboxsf utdelning monteringspunkt\&quot; för att komma åt den från ett operativsystem baserat på Linux. Denna funktion kräver Gästtillägg.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_th.ts	2018-12-29 18:11:46.871723184 +0100
@@ -6596,8 +6596,8 @@
         <translation type="vanished">อ่านอย่างเดียว</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>แสดงรายการโฟลเดอร์ใช้ร่วมกันทั้งหมดที่เข้าถึงได้จากเครื่องนี้ ใช้ &apos;net use x: \\vboxsvr \share&apos; เพื่อเข้าถึงโฟลเดอร์ใช้ร่วมกันชื่อ &lt;i&gt;share&lt;/i&gt; จากระบบปฏิบัติการแบบดอส หรือ &apos;mount -t vboxsf share mount_point&apos; เพื่อเข้าถึงจากระบบปฏิบัติการลินุกซ์ คุณสมบัตินี้ต้องใช้โปรแกรมเสริมสำหรับเกสต์</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>แสดงรายการโฟลเดอร์ใช้ร่วมกันทั้งหมดที่เข้าถึงได้จากเครื่องนี้ ใช้ &apos;net use x: \\vboxsvr \share&apos; เพื่อเข้าถึงโฟลเดอร์ใช้ร่วมกันชื่อ &lt;i&gt;share&lt;/i&gt; จากระบบปฏิบัติการแบบดอส หรือ &apos;mount -cit vboxsf share mount_point&apos; เพื่อเข้าถึงจากระบบปฏิบัติการลินุกซ์ คุณสมบัตินี้ต้องใช้โปรแกรมเสริมสำหรับเกสต์</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_tr.ts	2018-12-29 18:11:46.979723766 +0100
@@ -10050,8 +10050,8 @@
         <translation type="vanished">Salt-okunur</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Bu makineye erişilebilir tüm paylaşılan klasörleri listeler. DOS benzeri bir İS&apos;den &lt;i&gt;paylasim&lt;/i&gt; adındaki bir paylaşılan klasöre erişmek için &apos;net use x: \\vboxsvr\paylasim&apos; kullanın ya da bir Linux İS&apos;den bu klasöre erişmek için &apos;mount -t vboxsf paylasim mount_point&apos; kullanın. Bu özellik Misafir Eklentilerini gerektirir.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Bu makineye erişilebilir tüm paylaşılan klasörleri listeler. DOS benzeri bir İS&apos;den &lt;i&gt;paylasim&lt;/i&gt; adındaki bir paylaşılan klasöre erişmek için &apos;net use x: \\vboxsvr\paylasim&apos; kullanın ya da bir Linux İS&apos;den bu klasöre erişmek için &apos;mount -cit vboxsf paylasim mount_point&apos; kullanın. Bu özellik Misafir Eklentilerini gerektirir.</translation>
     </message>
     <message>
         <source>Name</source>
@@ -22310,12 +22310,12 @@
 Use
 &lt;tt&gt;net use x: \\vboxsvr\share&lt;/tt&gt;
 to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or
-&lt;tt&gt;mount -t vboxsf share mount_point&lt;/tt&gt;
+&lt;tt&gt;mount -cit vboxsf share mount_point&lt;/tt&gt;
 to access it from a Linux OS. This feature requires Guest Additions.&lt;/qt&gt;</source>
         <translation type="obsolete">&lt;qt&gt;Bu makinden erişilebilir tüm paylaşılan dizinleri listeler.
 DOS benzeri bir işletim sistemi üzerinde &lt;i&gt;paylasim&lt;/i&gt; olarak
 adlandırılan bir paylaşıma erişmek için &lt;tt&gt;net use x: \\vboxsvr\paylasim&lt;/tt&gt; kullanın ya da
-bir Linux makineden bu dizine erişebilmek için &lt;tt&gt;mount -t vboxsf paylasim mount_point&lt;/tt&gt; kullanın.
+bir Linux makineden bu dizine erişebilmek için &lt;tt&gt;mount -cit vboxsf paylasim mount_point&lt;/tt&gt; kullanın.
 Bu özellik Misafir Eklentilerinin yüklü olmasını gerektirir.&lt;/qt&gt;</translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_uk.ts	2018-12-29 18:11:46.845723044 +0100
@@ -9577,8 +9577,8 @@
         <translation type="vanished">Тільки для читання</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>Показує список всіх спільних тек, доступ до яких можливий з цього комп&apos;ютера. Скористайтеся командою &apos;net use x: \\vboxsvr\share&apos; DOS-подібної операційної системи або &apos;mount -t vboxsf share mount_point&apos; Linux-подібної, щоб отримати доступ до спільної теки &lt;i&gt;share&lt;/i&gt;. Для працездатності цієї можливості потрібні гостьові доповнення.</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>Показує список всіх спільних тек, доступ до яких можливий з цього комп&apos;ютера. Скористайтеся командою &apos;net use x: \\vboxsvr\share&apos; DOS-подібної операційної системи або &apos;mount -cit vboxsf share mount_point&apos; Linux-подібної, щоб отримати доступ до спільної теки &lt;i&gt;share&lt;/i&gt;. Для працездатності цієї можливості потрібні гостьові доповнення.</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_xx_YY.ts	2018-12-29 18:11:46.794722769 +0100
@@ -5258,7 +5258,7 @@
 <context>
     <name>UIMachineSettingsSF</name>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_CN.ts	2018-12-29 18:11:46.826722941 +0100
@@ -9350,8 +9350,8 @@
         <translation type="vanished">只读</translation>
     </message>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>列出本虚拟电脑可访问的所有共享文件夹。使用 &apos;net use x: \\vboxsvr\share&apos; 从DOS或Windows中访问名为: &lt;i&gt;share&lt;/i&gt; 的共享文件夹，或使用 &apos;mount -t vboxsf share mount_point&apos; 从Linux类的操作系统中访问该文件夹。该功能需要在虚拟电脑中安装增强功能包。</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>列出本虚拟电脑可访问的所有共享文件夹。使用 &apos;net use x: \\vboxsvr\share&apos; 从DOS或Windows中访问名为: &lt;i&gt;share&lt;/i&gt; 的共享文件夹，或使用 &apos;mount -cit vboxsf share mount_point&apos; 从Linux类的操作系统中访问该文件夹。该功能需要在虚拟电脑中安装增强功能包。</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts	2018-12-18 13:59:37.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/nls/VirtualBox_zh_TW.ts	2018-12-29 18:11:46.990723826 +0100
@@ -6062,8 +6062,8 @@
 <context>
     <name>UIMachineSettingsSF</name>
     <message>
-        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -t vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
-        <translation>列出所有可存取這部機器的共用資料夾。 從類似 DOS 的作業系統使用 [net use x: \\vboxsvr\share] 存取名為 &lt;i&gt;share&lt;/i&gt; 的共用資料夾，或從 Linux 作業系統 [mount -t vboxsf share mount_point] 來存取它。 這個功能需要 Guest Additions。</translation>
+        <source>Lists all shared folders accessible to this machine. Use &apos;net use x: \\vboxsvr\share&apos; to access a shared folder named &lt;i&gt;share&lt;/i&gt; from a DOS-like OS, or &apos;mount -cit vboxsf share mount_point&apos; to access it from a Linux OS. This feature requires Guest Additions.</source>
+        <translation>列出所有可存取這部機器的共用資料夾。 從類似 DOS 的作業系統使用 [net use x: \\vboxsvr\share] 存取名為 &lt;i&gt;share&lt;/i&gt; 的共用資料夾，或從 Linux 作業系統 [mount -cit vboxsf share mount_point] 來存取它。 這個功能需要 Guest Additions。</translation>
     </message>
     <message>
         <source>Name</source>
diff -Naur VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/settings/machine/UIMachineSettingsSF.ui VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/src/settings/machine/UIMachineSettingsSF.ui
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/settings/machine/UIMachineSettingsSF.ui	2018-12-18 13:59:46.000000000 +0100
+++ VirtualBox-6.0.0-mount/src/VBox/Frontends/VirtualBox/src/settings/machine/UIMachineSettingsSF.ui	2018-12-29 18:09:34.044006758 +0100
@@ -66,7 +66,7 @@
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="whatsThis">
-         <string>Lists all shared folders accessible to this machine. Use 'net use x: \\vboxsvr\share' to access a shared folder named &lt;i>share&lt;/i> from a DOS-like OS, or 'mount -t vboxsf share mount_point' to access it from a Linux OS. This feature requires Guest Additions.</string>
+         <string>Lists all shared folders accessible to this machine. Use 'net use x: \\vboxsvr\share' to access a shared folder named &lt;i>share&lt;/i> from a DOS-like OS, or 'mount -cit vboxsf share mount_point' to access it from a Linux OS. This feature requires Guest Additions.</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
