diff -urN VirtualBox-7.1.6/src/VBox/Additions/linux/drm/vbox_drv.c vbox/src/VBox/Additions/linux/drm/vbox_drv.c
--- VirtualBox-7.1.6/src/VBox/Additions/linux/drm/vbox_drv.c	2025-01-21 13:49:28.000000000 +0100
+++ vbox/src/VBox/Additions/linux/drm/vbox_drv.c	2025-03-25 21:40:16.380990696 +0100
@@ -84,6 +84,16 @@
 	struct drm_device *dev = NULL;
 	int ret = 0;
 
+#if RTLNX_VER_MIN(6,0,0)
+	static bool fWarned = false;
+	if (!fWarned)
+	{
+		printk(KERN_ERR "vboxvideo: VM is using legacy graphics controller, "
+				"please consider to configure this guest to use VMSVGA instead\n");
+		fWarned = true;
+	}
+#endif
+
 # if RTLNX_VER_RANGE(5,14,0, 6,13,0) || RTLNX_RHEL_RANGE(8,6, 8,99)
 #  if RTLNX_VER_MIN(5,15,0) || RTLNX_RHEL_RANGE(8,7, 8,99) || RTLNX_RHEL_MIN(9,1) || RTLNX_SUSE_MAJ_PREREQ(15,4)
 	ret = drm_aperture_remove_conflicting_pci_framebuffers(pdev, &driver);
@@ -379,7 +389,9 @@
 #endif
 	.name = DRIVER_NAME,
 	.desc = DRIVER_DESC,
+#if RTLNX_VER_MAX(6,14,0)
 	.date = DRIVER_DATE,
+#endif
 	.major = DRIVER_MAJOR,
 	.minor = DRIVER_MINOR,
 	.patchlevel = DRIVER_PATCHLEVEL,
