diff -up VirtualBox-6.0.8/src/VBox/Installer/linux/Makefile-header.gmk.0006~ VirtualBox-6.0.8/src/VBox/Installer/linux/Makefile-header.gmk
--- VirtualBox-6.0.8/src/VBox/Installer/linux/Makefile-header.gmk.0006~	2019-05-15 23:28:05.097588379 +0200
+++ VirtualBox-6.0.8/src/VBox/Installer/linux/Makefile-header.gmk	2019-05-15 23:28:59.191161633 +0200
@@ -181,17 +181,6 @@ else # neq($(KERNELRELEASE),)
  endif
 endif # neq($(KERNELRELEASE),)
 
-# Kernel build folder
-ifneq ($(shell if test -d $(KERN_DIR); then echo yes; fi),yes)
- $(error Error: unable to find the headers of the Linux kernel to build against (KERN_DIR=$(KERN_DIR)). \
-          Specify KERN_VER=<version> (currently $(KERN_VER)) and run Make again)
-endif
-# Kernel include folder
-KERN_INCL := $(KERN_DIR)/include
-# module install folder
-INSTALL_MOD_DIR ?= misc
-MODULE_DIR := $(INSTALL_MOD_PATH)/lib/modules/$(KERN_VER)/$(INSTALL_MOD_DIR)
-
 # For VBOX_GCC_CHECK_CC
 VBOX_CLOSEPAR := )
 VBOX_DOLLAR   := $$
@@ -208,28 +197,3 @@ VBOX_GCC_CHECK_CC = $(shell \
          esac; \
     else echo "$(3)"; fi; \
     rm -f /tmp/$(VBOX_DOLLAR)$(VBOX_DOLLAR).check.c; )
-
-#
-# Guess the module directory ASSUMING that this file is located in that directory.
-# Note! The special MAKEFILE_LIST variable was introduced in GNU make 3.80.
-#
-ifdef MAKEFILE_LIST
- VBOX_MODULE_SRC_DIR := $(dir $(word $(words $(MAKEFILE_LIST)),$(MAKEFILE_LIST)))
-else
- VBOX_MODULE_SRC_DIR := $(CURDIR)/
-endif
-
-
-# debug - show guesses.
-ifdef DEBUG
- ifndef VBOX_KERN_QUIET
-$(warning dbg: INSTALL_MOD_PATH = $(INSTALL_MOD_PATH))
-$(warning dbg: INSTALL_MOD_DIR  = $(INSTALL_MOD_DIR))
-$(warning dbg: KERN_DIR         = $(KERN_DIR))
-$(warning dbg: KERN_INCL        = $(KERN_INCL))
-$(warning dbg: KERN_VERSION     = $(KERN_VERSION))
-$(warning dbg: MODULE_DIR       = $(MODULE_DIR))
-$(warning dbg: VBOX_MODULE_SRC_DIR = $(VBOX_MODULE_SRC_DIR))
- endif
-endif
-
