diff -up VirtualBox-6.1.16/configure.omv~ VirtualBox-6.1.16/configure
--- VirtualBox-6.1.16/configure.omv~	2020-10-28 16:19:47.294143325 +0100
+++ VirtualBox-6.1.16/configure	2020-10-28 16:20:39.344672473 +0100
@@ -2028,7 +2028,7 @@ extern "C" int main(void)
 }
 EOF
   found=
-  SUPPYTHONLIBS="python2.7 python2.6 python3.1 python3.2 python3.3 python3.4 python3.4m python3.5 python3.5m python3.6 python3.6m python3.7 python3.7m python3.8 python3.8m"
+  SUPPYTHONLIBS="python2.7 python2.6 python3.1 python3.2 python3.3 python3.4 python3.4m python3.5 python3.5m python3.6 python3.6m python3.7 python3.7m python3.8 python3.8m python3.9"
   for p in $PYTHONDIR; do
     for d in $SUPPYTHONLIBS; do
       for b in lib/x86_64-linux-gnu lib/i386-linux-gnu lib64 lib/64 lib; do
