diff -up VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/net/UIUpdateDefs.cpp.0007~ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/net/UIUpdateDefs.cpp
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/net/UIUpdateDefs.cpp.0007~	2018-12-19 00:07:26.577757261 +0100
+++ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/net/UIUpdateDefs.cpp	2018-12-19 00:09:01.938043095 +0100
@@ -164,7 +164,7 @@ void VBoxUpdateData::decode()
             if (m_dayList.isEmpty())
                 populate();
             PeriodType index = (PeriodType)m_dayList.indexOf(VBoxUpdateDay(QString(), parser[0]));
-            m_enmPeriodIndex = index == PeriodUndefined ? Period1Day : index;
+            m_enmPeriodIndex = index == PeriodUndefined ? PeriodNever : index;
         }
 
         /* Parse 'date' value: */
diff -up VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/globals/UIActionPoolManager.cpp.omv~ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/globals/UIActionPoolManager.cpp
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/globals/UIActionPoolManager.cpp.omv~	2018-12-19 01:06:00.821689003 +0100
+++ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/globals/UIActionPoolManager.cpp	2018-12-19 01:06:16.471914386 +0100
@@ -2793,9 +2793,6 @@ void UIActionPoolManager::updateMenuFile
 # ifdef VBOX_GUI_WITH_NETWORK_MANAGER
     /* 'Network Access Manager' action goes to 'File' menu: */
     pMenu->addAction(action(UIActionIndex_M_Application_S_NetworkAccessManager));
-    /* 'Check for Updates' action goes to 'File' menu: */
-    if (gEDataManager->applicationUpdateEnabled())
-        pMenu->addAction(action(UIActionIndex_M_Application_S_CheckForUpdates));
 # endif
     /* Separator after tool actions of the 'File' menu: */
     pMenu->addSeparator();
diff -up VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/manager/chooser/UIChooserModel.cpp.omv~ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/manager/chooser/UIChooserModel.cpp
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/manager/chooser/UIChooserModel.cpp.omv~	2018-12-19 02:28:23.782235108 +0100
+++ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/manager/chooser/UIChooserModel.cpp	2018-12-19 02:28:28.850293782 +0100
@@ -1366,8 +1366,6 @@ void UIChooserModel::prepareContextMenu(
             m_pContextMenuGlobal->addAction(actionPool()->action(UIActionIndexST_M_File_S_ShowCloudProfileManager));
 # ifdef VBOX_GUI_WITH_NETWORK_MANAGER
         m_pContextMenuGlobal->addAction(actionPool()->action(UIActionIndex_M_Application_S_NetworkAccessManager));
-        if (gEDataManager->applicationUpdateEnabled())
-            m_pContextMenuGlobal->addAction(actionPool()->action(UIActionIndex_M_Application_S_CheckForUpdates));
 # endif
 #endif /* !VBOX_WS_MAC */
     }
