--- a/configure	2014-06-02 13:57:44.458605052 +0100
+++ b/configure	2014-06-02 13:57:54.498604465 +0100
@@ -420,7 +420,7 @@
       elif [ $cc_maj -lt 3 \
              -o \( $cc_maj -eq 3 -a $cc_min -lt 2 \) \
              -o \( $cc_maj -eq 4 -a $cc_min -lt 1 -a "$OS" != "darwin" \) \
-             -o \( $cc_maj -eq 4 -a $cc_min -gt 8 \) \
+             -o \( $cc_maj -eq 4 -a $cc_min -gt 9 \) \
              -o $cc_maj -gt 4 ]; then
         log_failure "gcc version $cc_ver found, expected gcc 3.x with x>1 or gcc 4.x with 0<x<8"
         fail really
