--- src/VBox/Devices/PC/Etherboot-src/Makefile.orig	2013-02-16 18:33:19.368337252 +0000
+++ src/VBox/Devices/PC/Etherboot-src/Makefile.kmk	2013-02-16 18:33:32.033337765 +0000
@@ -18,7 +18,7 @@
 SUB_DEPTH = ../../../../..
 include $(KBUILD_PATH)/subheader.kmk
 
-if1of ($(KBUILD_TARGET).$(KBUILD_TARGET_ARCH), darwin.x86 darwin.amd64 os2.x86 solaris.x86 solaris.amd64 win.x86 win.amd64 linux.amd64 freebsd.amd64)
+if1of ($(KBUILD_TARGET).$(KBUILD_TARGET_ARCH), darwin.x86 darwin.amd64 os2.x86 solaris.x86 solaris.amd64 win.x86 win.amd64 linux.amd64 linux.x86 freebsd.amd64)
  #
  # Build it from a precompiled binary.
  # See notes about the binary load address and size way below.
