diff -up VirtualBox-6.0.6/src/VBox/Installer/linux/VBox.sh.0001~ VirtualBox-6.0.6/src/VBox/Installer/linux/VBox.sh
--- VirtualBox-6.0.6/src/VBox/Installer/linux/VBox.sh.0001~	2019-04-16 12:16:46.000000000 +0200
+++ VirtualBox-6.0.6/src/VBox/Installer/linux/VBox.sh	2019-04-17 03:14:10.362183984 +0200
@@ -62,7 +62,7 @@ WARNING: The vboxdrv kernel module is no
          available for the current kernel (`uname -r`) or it failed to
          load. Please recompile the kernel module and install it by
 
-           sudo /sbin/vboxconfig
+           sudo dnf --refresh install kernel-release-desktop-latest
 
          You will not be able to start VMs until this problem is fixed.
 EOF
@@ -71,6 +71,8 @@ elif [ ! -c /dev/vboxdrv ]; then
 WARNING: The character device /dev/vboxdrv does not exist. Try
 
            sudo /sbin/vboxconfig
+           or
+           sudo dnf --refresh install kernel-release-desktop-latest
 
          and if that is not successful, try to re-install the package.
 
diff -up VirtualBox-6.0.6/src/VBox/VMM/VMMR3/VM.cpp.0001~ VirtualBox-6.0.6/src/VBox/VMM/VMMR3/VM.cpp
--- VirtualBox-6.0.6/src/VBox/VMM/VMMR3/VM.cpp.0001~	2019-04-17 03:14:10.363183984 +0200
+++ VirtualBox-6.0.6/src/VBox/VMM/VMMR3/VM.cpp	2019-04-17 03:18:48.068170511 +0200
@@ -274,9 +274,9 @@ VMMR3DECL(int)   VMR3Create(uint32_t cCp
                     pszError = N_("One of the kernel modules was not successfully loaded. Make sure "
                                   "that VirtualBox is correctly installed, and if you are using EFI "
                                   "Secure Boot that the modules are signed if necessary in the right "
-                                  "way for your host system.  Then try to recompile and reload the "
-                                  "kernel modules by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "way for your host system.  Then try to use the OpenMandriva "
+                                  "provided kernel modules by executing "
+                                  "'dnf --refresh install kernel-release-desktop' as root");
                     break;
 #endif
 
@@ -344,7 +344,7 @@ VMMR3DECL(int)   VMR3Create(uint32_t cCp
                                   "or you are using EFI Secure Boot and the module is not signed "
                                   "in the right way for your system.  If necessary, try setting up "
                                   "the kernel module again by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dnf --refresh install kernel-release-desktop' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not loaded");
 #endif
@@ -388,7 +388,7 @@ VMMR3DECL(int)   VMR3Create(uint32_t cCp
                                   "or you are using EFI Secure Boot and the module is not signed "
                                   "in the right way for your system.  If necessary, try setting up "
                                   "the kernel module again by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dnf --refresh install kernel-release-desktop' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not installed");
 #endif
