diff -up VirtualBox-5.2.10/configure.omv~ VirtualBox-5.2.10/configure
--- VirtualBox-5.2.10/configure.omv~	2018-04-27 00:06:59.430723900 +0200
+++ VirtualBox-5.2.10/configure	2018-04-27 00:07:12.610421993 +0200
@@ -440,8 +440,8 @@ check_gcc()
       elif [ $cc_maj -lt 4 \
              -o \( $cc_maj -eq 4 -a $cc_min -lt 4 -a "$OS" != "darwin" \) \
              -o \( $cc_maj -eq 4 -a $cc_min -lt 2 -a "$OS"  = "darwin" \) \
-             -o $cc_maj -gt 7 ]; then
-        log_failure "gcc version $cc_maj.$cc_min found, expected gcc 4.x...7.x"
+             -o $cc_maj -gt 8 ]; then
+        log_failure "gcc version $cc_maj.$cc_min found, expected gcc 4.x...8.x"
         fail really
       else
         log_success "found version $cc_ver"
