diff -Naur VirtualBox-5.1.0/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp VirtualBox-5.1.0.tpg/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp
--- VirtualBox-5.1.0/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2016-07-12 14:54:10.000000000 +0000
+++ VirtualBox-5.1.0.tpg/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2016-07-22 20:14:00.785904704 +0000
@@ -330,12 +330,12 @@
     /* Or select Ubuntu item for Linux family as default: */
     else if (strFamilyId == "Linux")
     {
-        QString strDefaultID = "Ubuntu";
+        QString strDefaultID = "Mandriva";
         if (ARCH_BITS == 64 && m_fSupportsHWVirtEx && m_fSupportsLongMode)
             strDefaultID += "_64";
-        const int iIndexUbuntu = m_pComboType->findData(strDefaultID, TypeID);
-        if (iIndexUbuntu != -1)
-            m_pComboType->setCurrentIndex(iIndexUbuntu);
+        const int iIndexMandriva = m_pComboType->findData(strDefaultID, TypeID);
+        if (iIndexMandriva != -1)
+            m_pComboType->setCurrentIndex(iIndexMandriva);
     }
     /* Else simply select the first one present: */
     else
