diff -Naur VirtualBox-5.1.0/src/VBox/Installer/linux/VBox.sh VirtualBox-5.1.0.tpg/src/VBox/Installer/linux/VBox.sh
--- VirtualBox-5.1.0/src/VBox/Installer/linux/VBox.sh	2016-07-12 14:54:27.000000000 +0000
+++ VirtualBox-5.1.0.tpg/src/VBox/Installer/linux/VBox.sh	2016-07-22 20:05:15.034670870 +0000
@@ -62,7 +62,7 @@
          available for the current kernel (`uname -r`) or it failed to
          load. Please recompile the kernel module and install it by
 
-           sudo /sbin/vboxconfig
+           sudo dnf --refresh install kernel-release-desktop-latest
 
          You will not be able to start VMs until this problem is fixed.
 EOF
@@ -71,6 +71,8 @@
 WARNING: The character device /dev/vboxdrv does not exist. Try
 
            sudo /sbin/vboxconfig
+           or
+           sudo dnf --refresh install kernel-release-desktop-latest
 
          and if that is not successful, try to re-install the package.
 
diff -Naur VirtualBox-5.1.0/src/VBox/VMM/VMMR3/VM.cpp VirtualBox-5.1.0.tpg/src/VBox/VMM/VMMR3/VM.cpp
--- VirtualBox-5.1.0/src/VBox/VMM/VMMR3/VM.cpp	2016-07-12 14:55:10.000000000 +0000
+++ VirtualBox-5.1.0.tpg/src/VBox/VMM/VMMR3/VM.cpp	2016-07-22 20:06:49.569237389 +0000
@@ -319,7 +319,7 @@
                     pszError = N_("One of the kernel modules was not successfully loaded. Make sure "
                                   "that no kernel modules from an older version of VirtualBox exist. "
-                                  "Then try to recompile and reload the kernel modules by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "Then try to install the latest kernel modules by executing "
+                                  "'dnf --refresh install kernel-release-desktop-latest' as root");
                     break;
 #endif
 
@@ -385,7 +385,7 @@
                     pszError = N_("VirtualBox kernel driver not loaded. The vboxdrv kernel module "
                                   "was either not loaded or /dev/vboxdrv is not set up properly. "
                                   "Re-setup the kernel module by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dnf --refresh install kernel-release-desktop-latest' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not loaded");
 #endif
@@ -427,7 +427,7 @@
                     pszError = N_("VirtualBox kernel driver not installed. The vboxdrv kernel module "
                                   "was either not loaded or /dev/vboxdrv was not created for some "
                                   "reason. Re-setup the kernel module by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dnf --refresh install kernel-release-desktop-latest' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not installed");
 #endif
