diff -up VirtualBox-7.0.8/src/VBox/Additions/x11/vboxvideo/Makefile.kmk.12~ VirtualBox-7.0.8/src/VBox/Additions/x11/vboxvideo/Makefile.kmk
--- VirtualBox-7.0.8/src/VBox/Additions/x11/vboxvideo/Makefile.kmk.12~	2023-04-20 20:49:29.676339991 +0200
+++ VirtualBox-7.0.8/src/VBox/Additions/x11/vboxvideo/Makefile.kmk	2023-04-20 21:01:47.691147821 +0200
@@ -406,7 +406,7 @@ vboxvideo_drv_118_LIBS += $(vboxvideo_dr
 
 ifdef VBOX_USE_SYSTEM_XORG_HEADERS
  # Build using local X.Org headers.  We assume X.Org Server 1.7 or later.
- DLLS    := $(filter-out vboxvideo_drv_%,$(DLLS)) vboxvideo_drv_system
+ DLLS    := $(filter-out vboxvideo_drv_%,$(DLLS))
  SYSMODS := $(filter-out vboxvideo_drv%,$(SYSMODS))
  vboxvideo_drv_system_TEMPLATE = VBoxGuestR3XOrgMod
  vboxvideo_drv_system_CFLAGS := \
