diff -up VirtualBox-6.1.16/Config.kmk.14~ VirtualBox-6.1.16/Config.kmk
--- VirtualBox-6.1.16/Config.kmk.14~	2020-10-16 18:27:46.000000000 +0200
+++ VirtualBox-6.1.16/Config.kmk	2020-10-28 14:47:22.514617605 +0100
@@ -4079,23 +4079,27 @@ VBOXLNX32GUEST_SUFF_LIB = .a
 # SDKs for external libraries.
 #
 SDK_VBOX_LIBXML2           = .
-SDK_VBOX_LIBXML2_INCS     ?= $(PATH_ROOT)/src/libs/libxml2-2.9.4/include
+SDK_VBOX_LIBXML2_INCS     ?= $(pkg-config libxml-2.0 --variable=includedir)
 SDK_VBOX_LIBXML2_DEFS     ?= _REENTRANT LIBXML_STATIC LIBXML_STATIC_FOR_DLL
 SDK_VBOX_LIBXML2_DEFS.win ?= WIN32 _WINDOWS _MBCS HAVE_WIN32_THREADS HAVE_COMPILER_TLS
 # Note: no linking to LIB here, we do that explicitly in src/VBox/Runtime/Makefile.kmk to link
 #       libxml against VBoxRT
 
 SDK_VBOX_LIBPNG       = .
-SDK_VBOX_LIBPNG_INCS ?= $(PATH_ROOT)/src/libs/libpng-1.6.36
-SDK_VBOX_LIBPNG_LIBS ?= $(PATH_STAGE_LIB)/VBox-libpng$(VBOX_SUFF_LIB)
+SDK_VBOX_LIBPNG_INCS ?= $(pkg-config libpng --variable=includedir)
+SDK_VBOX_LIBPNG_LIBS ?= $(pkg-config libpng --libs)
+
+SDK_VBOX_LIBLZF       = .
+SDK_VBOX_LIBLZF_INCS ?= $(pkg-config liblzf --variable=includedir)
+SDK_VBOX_LIBLZF_LIBS ?= $(pkg-config liblzf --libs)
 
 SDK_VBOX_ZLIB         = .
 if1of ($(KBUILD_TARGET), os2 solaris win)
  SDK_VBOX_ZLIB_INCS  ?= $(PATH_ROOT)/src/libs/zlib-1.2.11
  SDK_VBOX_ZLIB_LIBS  ?= $(PATH_STAGE_LIB)/VBox-zlib$(VBOX_SUFF_LIB)
 else
- SDK_VBOX_ZLIB_INCS  ?=
- SDK_VBOX_ZLIB_LIBS  ?= z
+ SDK_VBOX_ZLIB_INCS  ?= $(pkg-config zlib --variable=includedir)
+ SDK_VBOX_ZLIB_LIBS  ?= $(pkg-config zlib --libs)
  SDK_VBOX_ZLIB_DEFS.linux ?= _LARGEFILE64_SOURCE
 endif
 
@@ -4112,8 +4116,8 @@ if1of ($(KBUILD_TARGET), os2 win) # Not
 endif
 
 SDK_VBOX_LZF		= .
-SDK_VBOX_LZF_INCS  ?= $(PATH_ROOT)/src/libs/liblzf-3.4
-SDK_VBOX_LZF_LIBS  ?= $(PATH_STAGE_LIB)/VBox-liblzf$(VBOX_SUFF_LIB)
+SDK_VBOX_LZF_INCS  ?= $(pkg-config lzf --variable=includedir)
+SDK_VBOX_LZF_LIBS  ?= $(pkg-config lzf --libs)
 
 SDK_VBOX_LZF-x86	= .
 SDK_VBOX_LZF-x86_EXTENDS = VBOX_LZF
@@ -4158,28 +4162,28 @@ endif
 
 ifdef VBOX_WITH_LIBCURL
 SDK_VBOX_LIBCURL       = .
-SDK_VBOX_LIBCURL_INCS ?= $(PATH_ROOT)/src/libs/curl-7.64.0/include
-SDK_VBOX_LIBCURL_LIBS ?= $(PATH_STAGE_LIB)/VBox-libcurl$(VBOX_SUFF_LIB)
+SDK_VBOX_LIBCURL_INCS ?= $(pkg-config libcurl --variable=includedir)
+SDK_VBOX_LIBCURL_LIBS ?= $(pkg-config libcurl --libs)
 SDK_VBOX_LIBCURL_DEFS ?= BUILDING_LIBCURL
 
 SDK_VBOX_LIBCURL-x86           = .
-SDK_VBOX_LIBCURL-x86_INCS     ?= $(PATH_ROOT)/src/libs/curl-7.64.0/include
-SDK_VBOX_LIBCURL-x86_LIBS.x86 ?= $(PATH_STAGE_LIB)/VBox-libcurl-x86$(VBOX_SUFF_LIB)
+SDK_VBOX_LIBCURL-x86_INCS     ?= $(pkg-config libcurl --variable=includedir)
+SDK_VBOX_LIBCURL-x86_LIBS.x86 ?= $(pkg-config libcurl --libs)
 SDK_VBOX_LIBCURL-x86_DEFS     ?= BUILDING_LIBCURL
 endif
 
 ifdef VBOX_WITH_LIBVPX
  SDK_VBOX_VPX       = .
  SDK_VBOX_VPX_DEFAULT_INCS := $(PATH_ROOT)/src/libs/libvpx-1.6.1
- SDK_VBOX_VPX_INCS ?= $(SDK_VBOX_VPX_DEFAULT_INCS)
- SDK_VBOX_VPX_LIBS ?= $(PATH_STAGE_LIB)/VBox-libvpx$(VBOX_SUFF_LIB)
+ SDK_VBOX_VPX_INCS ?= $(pkg-config vpx --variable=includedir)
+ SDK_VBOX_VPX_LIBS ?= $(pkg-config vpx --libs)
 endif
 
 ifdef VBOX_WITH_LIBOPUS
  SDK_VBOX_OPUS       = .
  SDK_VBOX_OPUS_DEFAULT_INCS := $(PATH_ROOT)/src/libs/libopus-1.2.1/include
- SDK_VBOX_OPUS_INCS ?= $(SDK_VBOX_OPUS_DEFAULT_INCS)
- SDK_VBOX_OPUS_LIBS ?= $(PATH_STAGE_LIB)/VBox-libopus$(VBOX_SUFF_LIB)
+ SDK_VBOX_OPUS_INCS ?= $(pkg-config opus --variable=includedir)
+ SDK_VBOX_OPUS_LIBS ?= $(pkg-config opus --libs)
 endif
 
 ifdef VBOX_WITH_LIBFUSE
diff -up VirtualBox-6.1.16/src/libs/Makefile.kmk.14~ VirtualBox-6.1.16/src/libs/Makefile.kmk
diff -up VirtualBox-6.1.16/src/VBox/Runtime/Makefile.kmk.14~ VirtualBox-6.1.16/src/VBox/Runtime/Makefile.kmk
diff -up VirtualBox-6.1.16/src/VBox/Storage/testcase/Makefile.kmk.14~ VirtualBox-6.1.16/src/VBox/Storage/testcase/Makefile.kmk
diff -up VirtualBox-6.1.16/src/VBox/Runtime/common/zip/zip.cpp.omv~ VirtualBox-6.1.16/src/VBox/Runtime/common/zip/zip.cpp
--- VirtualBox-6.1.16/src/VBox/Runtime/common/zip/zip.cpp.omv~	2020-10-28 15:14:24.586267674 +0100
+++ VirtualBox-6.1.16/src/VBox/Runtime/common/zip/zip.cpp	2020-10-28 15:14:38.189440790 +0100
@@ -51,7 +51,9 @@
 # include <zlib.h>
 #endif
 #ifdef RTZIP_USE_LZF
+extern "C" {
 # include <lzf.h>
+}
 # include <iprt/crc.h>
 #endif
 #ifdef RTZIP_USE_LZJB
