--- VirtualBox-4.2.14/src/VBox/Additions/x11/vboxvideo/vboxvideo.c.omv~	2013-07-03 23:25:37.997284518 +0200
+++ VirtualBox-4.2.14/src/VBox/Additions/x11/vboxvideo/vboxvideo.c	2013-07-03 23:27:54.852215846 +0200
@@ -589,7 +589,7 @@ static XF86ModuleVersionInfo vboxVersion
 #ifdef XORG_7X
 _X_EXPORT
 #endif
-XF86ModuleData vboxvideoModuleData = { &vboxVersionRec, vboxSetup, NULL };
+__attribute__((used)) XF86ModuleData vboxvideoModuleData = { &vboxVersionRec, vboxSetup, NULL };
 
 static pointer
 vboxSetup(pointer Module, pointer Options, int *ErrorMajor, int *ErrorMinor)
