diff -up VirtualBox-7.0.16/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp.12~ VirtualBox-7.0.16/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp
--- VirtualBox-7.0.16/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp.12~	2024-04-17 14:33:26.496478608 +0200
+++ VirtualBox-7.0.16/src/VBox/Additions/common/VBoxGuest/lib/VBoxGuestR3LibRuntimeXF86.cpp	2024-04-17 15:27:20.345432280 +0200
@@ -43,22 +43,11 @@
 #include <iprt/assert.h>
 #include <iprt/log.h>
 #include <iprt/mem.h>
-#if defined(VBOX_VBGLR3_XFREE86)
-extern "C" {
-# define XFree86LOADER
-# if RT_GNUC_PREREQ(13,0) /* cmath gets dragged in and the c++/13/cmath header is allergic to -ffreestanding.  */
-#  define _GLIBCXX_INCLUDE_NEXT_C_HEADERS
-# endif
-# include <xf86_ansic.h>
-# undef size_t
-}
-#else
 # include <stdarg.h>
 # include <stdlib.h>
 # define xalloc malloc
 # define xfree free
 extern "C" void ErrorF(const char *f, ...);
-#endif
 
 RTDECL(void)    RTAssertMsg1Weak(const char *pszExpr, unsigned uLine, const char *pszFile, const char *pszFunction)
 {
diff -up VirtualBox-7.0.16/src/VBox/Additions/x11/vboxvideo/VBoxVideoIPRT.h.12~ VirtualBox-7.0.16/src/VBox/Additions/x11/vboxvideo/VBoxVideoIPRT.h
--- VirtualBox-7.0.16/src/VBox/Additions/x11/vboxvideo/VBoxVideoIPRT.h.12~	2024-04-16 13:58:16.000000000 +0200
+++ VirtualBox-7.0.16/src/VBox/Additions/x11/vboxvideo/VBoxVideoIPRT.h	2024-04-17 14:33:26.496478608 +0200
@@ -59,7 +59,7 @@ RT_C_DECLS_BEGIN
 RT_C_DECLS_END
 #endif  /* defined(IN_XF86_MODULE) && !defined(NO_ANSIC) */
 #define __STDC_LIMIT_MACROS  /* define *INT*_MAX on C++ too. */
-#include "compiler.h"  /* Can pull in <sdtint.h>.  Must come after xf86_ansic.h on XFree86. */
+#include <xorg/compiler.h>  /* Can pull in <stdint.h>.  Must come after xf86_ansic.h on XFree86. */
 #include <X11/Xfuncproto.h>
 #include <stdint.h>
 #if defined(IN_XF86_MODULE) && !defined(NO_ANSIC)
