diff -Naur VirtualBox-6.0.4/include/VBox/ostypes.h VirtualBox-6.0.4-p/include/VBox/ostypes.h
--- VirtualBox-6.0.4/include/VBox/ostypes.h	2019-01-25 19:10:05.000000000 +0100
+++ VirtualBox-6.0.4-p/include/VBox/ostypes.h	2019-01-31 18:16:30.057334709 +0100
@@ -99,8 +99,8 @@
     VBOXOSTYPE_FedoraCore_x64   = 0x57100,
     VBOXOSTYPE_Gentoo           = 0x58000,
     VBOXOSTYPE_Gentoo_x64       = 0x58100,
-    VBOXOSTYPE_Mandriva         = 0x59000,
-    VBOXOSTYPE_Mandriva_x64     = 0x59100,
+    VBOXOSTYPE_OpenMandriva     = 0x59000,
+    VBOXOSTYPE_OpenMandriva_x64 = 0x59100,
     VBOXOSTYPE_RedHat           = 0x5A000,
     VBOXOSTYPE_RedHat_x64       = 0x5A100,
     VBOXOSTYPE_Turbolinux       = 0x5B000,
diff -Naur VirtualBox-6.0.4/src/VBox/Additions/linux/installer/vboxadd-x11.sh VirtualBox-6.0.4-p/src/VBox/Additions/linux/installer/vboxadd-x11.sh
--- VirtualBox-6.0.4/src/VBox/Additions/linux/installer/vboxadd-x11.sh	2019-01-25 19:12:35.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Additions/linux/installer/vboxadd-x11.sh	2019-01-31 18:16:30.058334714 +0100
@@ -94,11 +94,11 @@
     x11_autostart="/etc/xdg/autostart"
     kde_autostart="/usr/share/autostart"
     redhat_dir=/etc/X11/Xsession.d
-    mandriva_dir=/etc/X11/xinit.d
+    openmandriva_dir=/etc/X11/xinit.d
     debian_dir=/etc/X11/xinit/xinitrc.d
-    if [ -d "$mandriva_dir" -a -w "$mandriva_dir" -a -x "$mandriva_dir" ]
+    if [ -d "$openmandriva_dir" -a -w "$openmandriva_dir" -a -x "$openmandriva_dir" ]
     then
-        install -m 0644 $app_src "$mandriva_dir/$app_dest"
+        install -m 0644 $app_src "$openmandriva_dir/$app_dest"
         found=1
     fi
     if [ -d "$x11_autostart" -a -w "$x11_autostart" -a -x "$x11_autostart" ]
diff -Naur VirtualBox-6.0.4/src/VBox/Devices/VMMDev/VMMDev.cpp VirtualBox-6.0.4-p/src/VBox/Devices/VMMDev/VMMDev.cpp
--- VirtualBox-6.0.4/src/VBox/Devices/VMMDev/VMMDev.cpp	2019-01-25 19:18:41.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Devices/VMMDev/VMMDev.cpp	2019-01-31 18:16:30.058334714 +0100
@@ -199,7 +199,7 @@
         case VBOXOSTYPE_OpenSUSE:                         pszOs = "openSUSE";       break;
         case VBOXOSTYPE_FedoraCore:                       pszOs = "Fedora";         break;
         case VBOXOSTYPE_Gentoo:                           pszOs = "Gentoo";         break;
-        case VBOXOSTYPE_Mandriva:                         pszOs = "Mandriva";       break;
+        case VBOXOSTYPE_OpenMandriva:                     pszOs = "OpenMandriva";   break;
         case VBOXOSTYPE_RedHat:                           pszOs = "RedHat";         break;
         case VBOXOSTYPE_Turbolinux:                       pszOs = "TurboLinux";     break;
         case VBOXOSTYPE_Ubuntu:                           pszOs = "Ubuntu";         break;
diff -Naur VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp
--- VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp	2019-01-25 19:19:38.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp	2019-01-31 18:16:30.059334719 +0100
@@ -325,8 +325,8 @@
     m_guestOSTypeIconNames.insert("Fedora_64",       ":/os_fedora_64.png");
     m_guestOSTypeIconNames.insert("Gentoo",          ":/os_gentoo.png");
     m_guestOSTypeIconNames.insert("Gentoo_64",       ":/os_gentoo_64.png");
-    m_guestOSTypeIconNames.insert("Mandriva",        ":/os_mandriva.png");
-    m_guestOSTypeIconNames.insert("Mandriva_64",     ":/os_mandriva_64.png");
+    m_guestOSTypeIconNames.insert("OpenMandriva",    ":/os_openmandriva.png");
+    m_guestOSTypeIconNames.insert("OpenMandriva_64", ":/os_openmandriva_64.png");
     m_guestOSTypeIconNames.insert("RedHat",          ":/os_redhat.png");
     m_guestOSTypeIconNames.insert("RedHat_64",       ":/os_redhat_64.png");
     m_guestOSTypeIconNames.insert("Turbolinux",      ":/os_turbolinux.png");
diff -Naur VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp
--- VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2019-01-25 19:19:46.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2019-01-31 18:16:30.059334719 +0100
@@ -245,15 +245,15 @@
         if (iIndexWin7 != -1)
             m_pComboType->setCurrentIndex(iIndexWin7);
     }
-    /* Or select Oracle Linux item for Linux family as default: */
+    /* Or select OpenMandriva Linux item for Linux family as default: */
     else if (m_strFamilyId == "Linux")
     {
-        QString strDefaultID = "Oracle";
+        QString strDefaultID = "OpenMandriva";
         if (ARCH_BITS == 64 && m_fSupportsHWVirtEx && m_fSupportsLongMode)
             strDefaultID += "_64";
-        const int iIndexUbuntu = m_pComboType->findData(strDefaultID, TypeID);
-        if (iIndexUbuntu != -1)
-            m_pComboType->setCurrentIndex(iIndexUbuntu);
+        const int iIndexOpenMandriva = m_pComboType->findData(strDefaultID, TypeID);
+        if (iIndexOpenMandriva != -1)
+            m_pComboType->setCurrentIndex(iIndexOpenMandriva);
     }
     /* Else simply select the first one present: */
     else
@@ -462,6 +462,10 @@
 
     /* Choose the 1st item to be the current: */
     m_pComboFamily->setCurrentIndex(0);
+    /* Default to Linux */
+    const int liIndex = m_pComboFamily->findData("Linux", TypeID);
+    if (liIndex != -1)
+        m_pComboFamily->setCurrentIndex(liIndex);
     /* And update the linked widgets accordingly: */
     sltFamilyChanged(m_pComboFamily->currentIndex());
 }
diff -Naur VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp
--- VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp	2019-01-25 19:19:48.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp	2019-01-31 18:16:30.059334719 +0100
@@ -137,8 +137,8 @@
     { QRegExp("Fe.*32",                            Qt::CaseInsensitive), "Fedora" },
     { QRegExp("((Gen)|(Sab)).*64",                 Qt::CaseInsensitive), "Gentoo_64" },
     { QRegExp("((Gen)|(Sab)).*32",                 Qt::CaseInsensitive), "Gentoo" },
-    { QRegExp("((Man)|(Mag)).*64",                 Qt::CaseInsensitive), "Mandriva_64" },
-    { QRegExp("((Man)|(Mag)).*32",                 Qt::CaseInsensitive), "Mandriva" },
+    { QRegExp("((Man)|(Mag)).*64",                 Qt::CaseInsensitive), "OpenMandriva_64" },
+    { QRegExp("((Man)|(Mag)).*32",                 Qt::CaseInsensitive), "OpenMandriva" },
     { QRegExp("((Red)|(rhel)|(cen)).*64",          Qt::CaseInsensitive), "RedHat_64" },
     { QRegExp("((Red)|(rhel)|(cen)).*32",          Qt::CaseInsensitive), "RedHat" },
     { QRegExp("Tur.*64",                           Qt::CaseInsensitive), "Turbolinux_64" },
diff -Naur VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/VirtualBox2.qrc VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/VirtualBox2.qrc
--- VirtualBox-6.0.4/src/VBox/Frontends/VirtualBox/VirtualBox2.qrc	2019-01-25 19:19:19.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Frontends/VirtualBox/VirtualBox2.qrc	2019-01-31 18:16:30.059334719 +0100
@@ -55,8 +55,8 @@
         <file alias="os_linux.png">images/os_linux.png</file>
         <file alias="os_macosx_64.png">images/os_macosx_64.png</file>
         <file alias="os_macosx.png">images/os_macosx.png</file>
-        <file alias="os_mandriva_64.png">images/os_mandriva_64.png</file>
-        <file alias="os_mandriva.png">images/os_mandriva.png</file>
+        <file alias="os_openmandriva_64.png">images/os_openmandriva_64.png</file>
+        <file alias="os_openmandriva.png">images/os_openmandriva.png</file>
         <file alias="os_netbsd_64.png">images/os_netbsd_64.png</file>
         <file alias="os_netbsd.png">images/os_netbsd.png</file>
         <file alias="os_netware.png">images/os_netware.png</file>
diff -Naur VirtualBox-6.0.4/src/VBox/Installer/linux/distributions_rpm VirtualBox-6.0.4-p/src/VBox/Installer/linux/distributions_rpm
--- VirtualBox-6.0.4/src/VBox/Installer/linux/distributions_rpm	2019-01-25 19:20:05.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Installer/linux/distributions_rpm	2019-01-31 18:16:30.059334719 +0100
@@ -7,6 +7,9 @@
 sles11.0        = SLES_11_0
 sles10.1        = SLES_10_1
 mdv2011.0       = MANDRIVA_2011_0
+omv2014.0       = OPENMANDRIVA_2014_0
+omv2015.0       = OPENMANDRIVA_Lx3
+omv4000         = OPENMANDRIVA_Lx4
 fedora29        = FEDORA_29
 fedora26        = FEDORA_26
 fedora25        = FEDORA_25
diff -Naur VirtualBox-6.0.4/src/VBox/Installer/linux/rpm/rules VirtualBox-6.0.4-p/src/VBox/Installer/linux/rpm/rules
--- VirtualBox-6.0.4/src/VBox/Installer/linux/rpm/rules	2019-01-25 19:20:05.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Installer/linux/rpm/rules	2019-01-31 18:16:30.060334724 +0100
@@ -47,6 +47,9 @@
 ifeq ($(wildcard rpm/rules),)
 $(error call rpm/rules from src/VBox/Installer/linux)
 endif
+ifeq ($(shell if grep -q '^%distsuffix' /usr/lib/rpm/macros.d/macros.OpenMandriva 2>/dev/null; then echo yes; fi),yes)
+$(error Fix /usr/lib/rpm/macros.d/macros.OpenMandriva)
+endif
 
 verpkg   := VirtualBox-6.0
 current  := $(shell pwd)
diff -Naur VirtualBox-6.0.4/src/VBox/Installer/linux/VBoxSysInfo.sh VirtualBox-6.0.4-p/src/VBox/Installer/linux/VBoxSysInfo.sh
--- VirtualBox-6.0.4/src/VBox/Installer/linux/VBoxSysInfo.sh	2019-01-25 19:20:04.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Installer/linux/VBoxSysInfo.sh	2019-01-31 18:16:30.060334724 +0100
@@ -44,11 +44,11 @@
         # Debian-based system
         release=`cat /etc/debian_version`
         print_linux_info "Debian" $release
-    elif [ -r /etc/mandriva-release ]
+    elif [ -r /etc/openmandriva-release ]
     then
-        # Mandriva-based system
-        release=`cat /etc/mandriva-release | sed -e 's/[A-Za-z ]* release //'`
-        print_linux_info "Mandriva" $release
+        # OpenMandriva-based system
+        release=`cat /etc/openmandriva-release | sed -e 's/[A-Za-z ]* release //'`
+        print_linux_info "OpenMandriva" $release
     elif [ -r /etc/fedora-release ]
     then
         # Fedora-based
diff -Naur VirtualBox-6.0.4/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h VirtualBox-6.0.4-p/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h
--- VirtualBox-6.0.4/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h	2019-01-25 19:20:07.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h	2019-01-31 18:16:30.060334724 +0100
@@ -911,8 +911,8 @@
     CIMOSType_CIMOS_NovellOES = 86,
     CIMOSType_CIMOS_NovellLinuxDesktop = 87,
     CIMOSType_CIMOS_SunJavaDesktopSystem = 88,
-    CIMOSType_CIMOS_Mandriva = 89,
-    CIMOSType_CIMOS_Mandriva_64 = 90,
+    CIMOSType_CIMOS_OpenMandriva = 89,
+    CIMOSType_CIMOS_OpenMandriva_64 = 90,
     CIMOSType_CIMOS_TurboLinux = 91,
     CIMOSType_CIMOS_TurboLinux_64 = 92,
     CIMOSType_CIMOS_Ubuntu = 93,
diff -Naur VirtualBox-6.0.4/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h VirtualBox-6.0.4-p/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h
--- VirtualBox-6.0.4/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h	2019-01-25 19:20:07.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h	2019-01-31 18:16:30.061334730 +0100
@@ -1361,8 +1361,8 @@
     CIMOSType_CIMOS_NovellOES = 86,
     CIMOSType_CIMOS_NovellLinuxDesktop = 87,
     CIMOSType_CIMOS_SunJavaDesktopSystem = 88,
-    CIMOSType_CIMOS_Mandriva = 89,
-    CIMOSType_CIMOS_Mandriva_64 = 90,
+    CIMOSType_CIMOS_OpenMandriva = 89,
+    CIMOSType_CIMOS_OpenMandriva_64 = 90,
     CIMOSType_CIMOS_TurboLinux = 91,
     CIMOSType_CIMOS_TurboLinux_64 = 92,
     CIMOSType_CIMOS_Ubuntu = 93,
diff -Naur VirtualBox-6.0.4/src/VBox/Main/include/ovfreader.h VirtualBox-6.0.4-p/src/VBox/Main/include/ovfreader.h
--- VirtualBox-6.0.4/src/VBox/Main/include/ovfreader.h	2019-01-25 19:20:10.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Main/include/ovfreader.h	2019-01-31 18:16:30.061334730 +0100
@@ -156,8 +156,8 @@
     CIMOSType_CIMOS_NovellOES   = 86,
     CIMOSType_CIMOS_NovellLinuxDesktop  = 87,
     CIMOSType_CIMOS_SunJavaDesktopSystem    = 88,
-    CIMOSType_CIMOS_Mandriva    = 89,
-    CIMOSType_CIMOS_Mandriva_64 = 90,
+    CIMOSType_CIMOS_OpenMandriva    = 89,
+    CIMOSType_CIMOS_OpenMandriva_64 = 90,
     CIMOSType_CIMOS_TurboLinux  = 91,
     CIMOSType_CIMOS_TurboLinux_64   = 92,
     CIMOSType_CIMOS_Ubuntu  = 93,
diff -Naur VirtualBox-6.0.4/src/VBox/Main/src-all/Global.cpp VirtualBox-6.0.4-p/src/VBox/Main/src-all/Global.cpp
--- VirtualBox-6.0.4/src/VBox/Main/src-all/Global.cpp	2019-01-25 19:20:10.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Main/src-all/Global.cpp	2019-01-31 18:18:20.737908072 +0100
@@ -251,15 +251,15 @@
       1024,  16,  8 * _1G64, GraphicsControllerType_VMSVGA, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
         StorageControllerType_IntelAhci, StorageBus_SATA, ChipsetType_PIIX3, AudioControllerType_AC97, AudioCodecType_AD1980  },
 
-    { "Linux",   "Linux",             "Mandriva",           "Mandriva (32-bit)",
-      VBOXOSTYPE_Mandriva,        VBOXOSHINT_RTCUTC | VBOXOSHINT_USBTABLET | VBOXOSHINT_X2APIC,
-      1024,  16,  8 * _1G64, GraphicsControllerType_VMSVGA, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
+    { "Linux",   "Linux",             "OpenMandriva",           "OpenMandriva (32-bit)",
+      VBOXOSTYPE_OpenMandriva,        VBOXOSHINT_RTCUTC | VBOXOSHINT_USBTABLET | VBOXOSHINT_X2APIC,
+      2048,  16,  8 * _1G64, GraphicsControllerType_VMSVGA, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
         StorageControllerType_IntelAhci, StorageBus_SATA, ChipsetType_PIIX3, AudioControllerType_AC97, AudioCodecType_AD1980  },
 
-    { "Linux",   "Linux",             "Mandriva_64",        "Mandriva (64-bit)",
-      VBOXOSTYPE_Mandriva_x64,    VBOXOSHINT_64BIT | VBOXOSHINT_HWVIRTEX | VBOXOSHINT_IOAPIC | VBOXOSHINT_RTCUTC
+    { "Linux",   "Linux",             "OpenMandriva_64",        "OpenMandriva (64-bit)",
+      VBOXOSTYPE_OpenMandriva_x64,    VBOXOSHINT_64BIT | VBOXOSHINT_HWVIRTEX | VBOXOSHINT_IOAPIC | VBOXOSHINT_RTCUTC
                                 | VBOXOSHINT_USBTABLET | VBOXOSHINT_X2APIC,
-      1024,  16,  8 * _1G64, GraphicsControllerType_VMSVGA, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
+      2048,  16,  8 * _1G64, GraphicsControllerType_VMSVGA, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
         StorageControllerType_IntelAhci, StorageBus_SATA, ChipsetType_PIIX3, AudioControllerType_AC97, AudioCodecType_AD1980  },
 
     { "Linux",   "Linux",             "Oracle",             "Oracle (32-bit)",
diff -Naur VirtualBox-6.0.4/src/VBox/Main/src-server/ApplianceImpl.cpp VirtualBox-6.0.4-p/src/VBox/Main/src-server/ApplianceImpl.cpp
--- VirtualBox-6.0.4/src/VBox/Main/src-server/ApplianceImpl.cpp	2019-01-25 19:20:12.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Main/src-server/ApplianceImpl.cpp	2019-01-31 18:16:30.062334735 +0100
@@ -137,8 +137,8 @@
     { ovf::CIMOSType_CIMOS_SunJavaDesktopSystem,                 VBOXOSTYPE_Linux },
     { ovf::CIMOSType_CIMOS_TurboLinux,                           VBOXOSTYPE_Turbolinux },
     { ovf::CIMOSType_CIMOS_TurboLinux_64,                        VBOXOSTYPE_Turbolinux_x64 },
-    { ovf::CIMOSType_CIMOS_Mandriva,                             VBOXOSTYPE_Mandriva },
-    { ovf::CIMOSType_CIMOS_Mandriva_64,                          VBOXOSTYPE_Mandriva_x64 },
+    { ovf::CIMOSType_CIMOS_OpenMandriva,                         VBOXOSTYPE_OpenMandriva },
+    { ovf::CIMOSType_CIMOS_OpenMandriva_64,                      VBOXOSTYPE_OpenMandriva_x64 },
     { ovf::CIMOSType_CIMOS_Ubuntu,                               VBOXOSTYPE_Ubuntu },
     { ovf::CIMOSType_CIMOS_Ubuntu_64,                            VBOXOSTYPE_Ubuntu_x64 },
     { ovf::CIMOSType_CIMOS_Debian,                               VBOXOSTYPE_Debian },
@@ -200,7 +200,7 @@
     {"SUSE",          VBOXOSTYPE_OpenSUSE},
     {"Novell",        VBOXOSTYPE_OpenSUSE},
     {"Red Hat",       VBOXOSTYPE_RedHat},
-    {"Mandriva",      VBOXOSTYPE_Mandriva},
+    {"OpenMandriva",  VBOXOSTYPE_OpenMandriva},
     {"Ubuntu",        VBOXOSTYPE_Ubuntu},
     {"Debian",        VBOXOSTYPE_Debian},
     {"QNX",           VBOXOSTYPE_QNX},
@@ -229,7 +229,7 @@
     {"SUSE",          VBOXOSTYPE_OpenSUSE_x64},
     {"Novell",        VBOXOSTYPE_OpenSUSE_x64},
     {"Red Hat",       VBOXOSTYPE_RedHat_x64},
-    {"Mandriva",      VBOXOSTYPE_Mandriva_x64},
+    {"OpenMandriva",  VBOXOSTYPE_OpenMandriva_x64},
     {"Ubuntu",        VBOXOSTYPE_Ubuntu_x64},
     {"Debian",        VBOXOSTYPE_Debian_x64},
     {"Linux 2.4",     VBOXOSTYPE_Linux24_x64},
diff -Naur VirtualBox-6.0.4/src/VBox/Main/xml/Settings.cpp VirtualBox-6.0.4-p/src/VBox/Main/xml/Settings.cpp
--- VirtualBox-6.0.4/src/VBox/Main/xml/Settings.cpp	2019-01-25 19:20:16.000000000 +0100
+++ VirtualBox-6.0.4-p/src/VBox/Main/xml/Settings.cpp	2019-01-31 18:16:30.063334740 +0100
@@ -5399,7 +5399,7 @@
     { "opensuse", "OpenSUSE" },
     { "fedoracore", "Fedora" },
     { "gentoo", "Gentoo" },
-    { "mandriva", "Mandriva" },
+    { "openmandriva", "OpenMandriva" },
     { "redhat", "RedHat" },
     { "ubuntu", "Ubuntu" },
     { "xandros", "Xandros" },
