--- VirtualBox-4.3.6/src/VBox/Additions/x11/vboxvideo/Makefile.kmk.omv~	2014-01-07 20:04:54.055844804 +0100
+++ VirtualBox-4.3.6/src/VBox/Additions/x11/vboxvideo/Makefile.kmk	2014-01-07 20:05:38.872512062 +0100
@@ -333,6 +333,20 @@ vboxvideo_drv_114_INCS += $(PATH_ROOT)/s
 vboxvideo_drv_114_SOURCES := $(vboxvideo_drv_15_SOURCES)
 
 
+#
+# vboxvideo_drv_115
+#
+DLLS += vboxvideo_drv_115
+vboxvideo_drv_115_TEMPLATE = VBOXGUESTR3XORGMOD
+vboxvideo_drv_115_CFLAGS := $(vboxvideo_drv_70_CFLAGS)
+vboxvideo_drv_115_DEFS := $(vboxvideo_15_DEFS) XORG_VERSION_CURRENT=101400000
+vboxvideo_drv_115_INCS = \
+	$(vboxvideo_xorg_INCS) \
+	/usr/include/xorg
+vboxvideo_drv_115_INCS += $(PATH_ROOT)/src/VBox/Runtime/include
+vboxvideo_drv_115_SOURCES := $(vboxvideo_drv_15_SOURCES)
+
+
 # 32-bit modules for the 64-bit Linux Additions
 ifeq ($(KBUILD_TARGET).$(KBUILD_TARGET_ARCH),linux.amd64)
  ifneq ($(VBOX_WITH_COMPATIBLE_LINUX_GUEST_PACKAGE),)
@@ -399,7 +413,7 @@ $$(vboxvideo_drv_0_OUTDIR)/tstvboxvideo6
 	$$(QUIET)$$(APPEND) -t "$$@" "done"
     endef
 
-    $(foreach ver, _70 _71 _13 _14 _15 _16 _17 _18 _19 _110 _111 _112 _113 _114, $(eval $(def_vboxvideo_test)))
+    $(foreach ver, _70 _71 _13 _14 _15 _16 _17 _18 _19 _110 _111 _112 _113 _114 _115, $(eval $(def_vboxvideo_test)))
 
     endif # ! VBOX_ONLY_SDK
    endif # eq ($(KBUILD_HOST_ARCH),$(KBUILD_TARGET_ARCH))
