diff -Naur VirtualBox-5.2.12/include/VBox/ostypes.h VirtualBox-5.2.12.tpg/include/VBox/ostypes.h
--- VirtualBox-5.2.12/include/VBox/ostypes.h	2018-05-09 16:32:03.000000000 +0000
+++ VirtualBox-5.2.12.tpg/include/VBox/ostypes.h	2018-06-15 12:44:53.607162147 +0000
@@ -95,8 +95,8 @@
     VBOXOSTYPE_FedoraCore_x64   = 0x57100,
     VBOXOSTYPE_Gentoo           = 0x58000,
     VBOXOSTYPE_Gentoo_x64       = 0x58100,
-    VBOXOSTYPE_Mandriva         = 0x59000,
-    VBOXOSTYPE_Mandriva_x64     = 0x59100,
+    VBOXOSTYPE_OpenMandriva     = 0x59000,
+    VBOXOSTYPE_OpenMandriva_x64 = 0x59100,
     VBOXOSTYPE_RedHat           = 0x5A000,
     VBOXOSTYPE_RedHat_x64       = 0x5A100,
     VBOXOSTYPE_Turbolinux       = 0x5B000,
diff -Naur VirtualBox-5.2.12/src/VBox/Additions/linux/installer/vboxadd-x11.sh VirtualBox-5.2.12.tpg/src/VBox/Additions/linux/installer/vboxadd-x11.sh
--- VirtualBox-5.2.12/src/VBox/Additions/linux/installer/vboxadd-x11.sh	2018-05-09 16:32:19.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Additions/linux/installer/vboxadd-x11.sh	2018-06-15 12:44:53.608162147 +0000
@@ -94,11 +94,11 @@
     x11_autostart="/etc/xdg/autostart"
     kde_autostart="/usr/share/autostart"
     redhat_dir=/etc/X11/Xsession.d
-    mandriva_dir=/etc/X11/xinit.d
+    openmandriva_dir=/etc/X11/xinit.d
     debian_dir=/etc/X11/xinit/xinitrc.d
-    if [ -d "$mandriva_dir" -a -w "$mandriva_dir" -a -x "$mandriva_dir" ]
+    if [ -d "$openmandriva_dir" -a -w "$openmandriva_dir" -a -x "$openmandriva_dir" ]
     then
-        install -m 0644 $app_src "$mandriva_dir/$app_dest"
+        install -m 0644 $app_src "$openmandriva_dir/$app_dest"
         found=1
     fi
     if [ -d "$x11_autostart" -a -w "$x11_autostart" -a -x "$x11_autostart" ]
diff -Naur VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/VirtualBox1.qrc VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/VirtualBox1.qrc
--- VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/VirtualBox1.qrc	2018-05-09 16:37:50.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/VirtualBox1.qrc	2018-06-15 12:44:53.608162147 +0000
@@ -290,8 +290,8 @@
         <file alias="os_linux26_64.png">images/os_linux26_64.png</file>
         <file alias="os_macosx.png">images/os_macosx.png</file>
         <file alias="os_macosx_64.png">images/os_macosx_64.png</file>
-        <file alias="os_mandriva.png">images/os_mandriva.png</file>
-        <file alias="os_mandriva_64.png">images/os_mandriva_64.png</file>
+        <file alias="os_openmandriva.png">images/os_openmandriva.png</file>
+        <file alias="os_openmandriva_64.png">images/os_openmandriva_64.png</file>
         <file alias="os_netbsd.png">images/os_netbsd.png</file>
         <file alias="os_netbsd_64.png">images/os_netbsd_64.png</file>
         <file alias="os_netware.png">images/os_netware.png</file>
diff -Naur VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp
--- VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp	2018-05-09 16:37:58.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/src/globals/UIIconPool.cpp	2018-06-15 12:44:53.608162147 +0000
@@ -316,8 +316,8 @@
     m_guestOSTypeIconNames.insert("Fedora_64",       ":/os_fedora_64.png");
     m_guestOSTypeIconNames.insert("Gentoo",          ":/os_gentoo.png");
     m_guestOSTypeIconNames.insert("Gentoo_64",       ":/os_gentoo_64.png");
-    m_guestOSTypeIconNames.insert("Mandriva",        ":/os_mandriva.png");
-    m_guestOSTypeIconNames.insert("Mandriva_64",     ":/os_mandriva_64.png");
+    m_guestOSTypeIconNames.insert("OpenMandriva",        ":/os_openmandriva.png");
+    m_guestOSTypeIconNames.insert("OpenMandriva_64",     ":/os_openmandriva_64.png");
     m_guestOSTypeIconNames.insert("RedHat",          ":/os_redhat.png");
     m_guestOSTypeIconNames.insert("RedHat_64",       ":/os_redhat_64.png");
     m_guestOSTypeIconNames.insert("Turbolinux",      ":/os_turbolinux.png");
diff -Naur VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp
--- VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2018-06-15 12:44:16.375160000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2018-06-15 12:46:14.661165236 +0000
@@ -213,6 +213,10 @@
 
     /* Choose the 1st item to be the current: */
     m_pComboFamily->setCurrentIndex(0);
+    /* Default to Linux */
+    const int liIndex = m_pComboFamily->findData("Linux", TypeID);
+    if (liIndex != -1)
+        m_pComboFamily->setCurrentIndex(liIndex);
     /* And update the linked widgets accordingly: */
     sltFamilyChanged(m_pComboFamily->currentIndex());
 }
@@ -272,6 +276,7 @@
         if (iTypeIndex != -1)
             m_pComboType->setCurrentIndex(iTypeIndex);
     }
+
 }
 
 void UINameAndSystemEditor::retranslateUi()
@@ -317,6 +322,16 @@
         if (iTypeIndex != -1)
             m_pComboType->setCurrentIndex(iTypeIndex);
     }
+    /* Or select OpenMandriva item for Linux family as default: */
+    else if (strFamilyId == "Linux")
+    {
+        QString strDefaultID = "OpenMandriva";
+        if (ARCH_BITS == 64 && m_fSupportsHWVirtEx && m_fSupportsLongMode)
+            strDefaultID += "_64";
+        const int iIndexOpenMandriva = m_pComboType->findData(strDefaultID, TypeID);
+        if (iIndexOpenMandriva != -1)
+            m_pComboType->setCurrentIndex(iIndexOpenMandriva);
+    }
     /* Or select Windows 7 item for Windows family as default: */
     else if (strFamilyId == "Windows")
     {
@@ -327,16 +342,6 @@
         if (iIndexWin7 != -1)
             m_pComboType->setCurrentIndex(iIndexWin7);
     }
-    /* Or select Ubuntu item for Linux family as default: */
-    else if (strFamilyId == "Linux")
-    {
-        QString strDefaultID = "Mandriva";
-        if (ARCH_BITS == 64 && m_fSupportsHWVirtEx && m_fSupportsLongMode)
-            strDefaultID += "_64";
-        const int iIndexMandriva = m_pComboType->findData(strDefaultID, TypeID);
-        if (iIndexMandriva != -1)
-            m_pComboType->setCurrentIndex(iIndexMandriva);
-    }
     /* Else simply select the first one present: */
     else
         m_pComboType->setCurrentIndex(0);
diff -Naur VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp
--- VirtualBox-5.2.12/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp	2018-05-09 16:38:11.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Frontends/VirtualBox/src/wizards/newvm/UIWizardNewVMPageBasic1.cpp	2018-06-15 12:44:53.608162147 +0000
@@ -138,8 +138,8 @@
     { QRegExp("Fe.*32",                            Qt::CaseInsensitive), "Fedora" },
     { QRegExp("((Gen)|(Sab)).*64",                 Qt::CaseInsensitive), "Gentoo_64" },
     { QRegExp("((Gen)|(Sab)).*32",                 Qt::CaseInsensitive), "Gentoo" },
-    { QRegExp("((Man)|(Mag)).*64",                 Qt::CaseInsensitive), "Mandriva_64" },
-    { QRegExp("((Man)|(Mag)).*32",                 Qt::CaseInsensitive), "Mandriva" },
+    { QRegExp("((Man)|(Mag)).*64",                 Qt::CaseInsensitive), "OpenMandriva_64" },
+    { QRegExp("((Man)|(Mag)).*32",                 Qt::CaseInsensitive), "OpenMandriva" },
     { QRegExp("((Red)|(rhel)|(cen)).*64",          Qt::CaseInsensitive), "RedHat_64" },
     { QRegExp("((Red)|(rhel)|(cen)).*32",          Qt::CaseInsensitive), "RedHat" },
     { QRegExp("Tur.*64",                           Qt::CaseInsensitive), "Turbolinux_64" },
diff -Naur VirtualBox-5.2.12/src/VBox/Installer/linux/VBoxSysInfo.sh VirtualBox-5.2.12.tpg/src/VBox/Installer/linux/VBoxSysInfo.sh
--- VirtualBox-5.2.12/src/VBox/Installer/linux/VBoxSysInfo.sh	2018-05-09 16:38:25.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Installer/linux/VBoxSysInfo.sh	2018-06-15 12:44:53.609162147 +0000
@@ -44,11 +44,11 @@
         # Debian-based system
         release=`cat /etc/debian_version`
         print_linux_info "Debian" $release
-    elif [ -r /etc/mandriva-release ]
+    elif [ -r /etc/openmandriva-release ]
     then
-        # Mandriva-based system
-        release=`cat /etc/mandriva-release | sed -e 's/[A-Za-z ]* release //'`
-        print_linux_info "Mandriva" $release
+        # OpenMandriva-based system
+        release=`cat /etc/openmandriva-release | sed -e 's/[A-Za-z ]* release //'`
+        print_linux_info "OpenMandriva" $release
     elif [ -r /etc/fedora-release ]
     then
         # Fedora-based
diff -Naur VirtualBox-5.2.12/src/VBox/Installer/linux/distributions_rpm VirtualBox-5.2.12.tpg/src/VBox/Installer/linux/distributions_rpm
--- VirtualBox-5.2.12/src/VBox/Installer/linux/distributions_rpm	2018-05-09 16:38:25.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Installer/linux/distributions_rpm	2018-06-15 12:44:53.609162147 +0000
@@ -6,6 +6,9 @@
 sles11.0        = SLES_11_0
 sles10.1        = SLES_10_1
 mdv2011.0       = MANDRIVA_2011_0
+omv2014.0       = OPENMANDRIVA_2014_0
+omv2015.0       = OPENMANDRIVA_Lx3
+omv4000         = OPENMANDRIVA_Lx4
 fedora26        = FEDORA_26
 fedora25        = FEDORA_25
 fedora24        = FEDORA_24
diff -Naur VirtualBox-5.2.12/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h VirtualBox-5.2.12.tpg/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h
--- VirtualBox-5.2.12/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h	2018-05-09 16:38:27.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Main/cbinding/VBoxCAPI_v2_2.h	2018-06-15 12:44:53.609162147 +0000
@@ -908,8 +908,8 @@
     CIMOSType_CIMOS_NovellOES = 86,
     CIMOSType_CIMOS_NovellLinuxDesktop = 87,
     CIMOSType_CIMOS_SunJavaDesktopSystem = 88,
-    CIMOSType_CIMOS_Mandriva = 89,
-    CIMOSType_CIMOS_Mandriva_64 = 90,
+    CIMOSType_CIMOS_OpenMandriva = 89,
+    CIMOSType_CIMOS_OpenMandriva_64 = 90,
     CIMOSType_CIMOS_TurboLinux = 91,
     CIMOSType_CIMOS_TurboLinux_64 = 92,
     CIMOSType_CIMOS_Ubuntu = 93,
diff -Naur VirtualBox-5.2.12/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h VirtualBox-5.2.12.tpg/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h
--- VirtualBox-5.2.12/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h	2018-05-09 16:38:27.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Main/cbinding/VBoxCAPI_v3_0.h	2018-06-15 12:44:53.610162147 +0000
@@ -1358,8 +1358,8 @@
     CIMOSType_CIMOS_NovellOES = 86,
     CIMOSType_CIMOS_NovellLinuxDesktop = 87,
     CIMOSType_CIMOS_SunJavaDesktopSystem = 88,
-    CIMOSType_CIMOS_Mandriva = 89,
-    CIMOSType_CIMOS_Mandriva_64 = 90,
+    CIMOSType_CIMOS_OpenMandriva = 89,
+    CIMOSType_CIMOS_OpenMandriva_64 = 90,
     CIMOSType_CIMOS_TurboLinux = 91,
     CIMOSType_CIMOS_TurboLinux_64 = 92,
     CIMOSType_CIMOS_Ubuntu = 93,
diff -Naur VirtualBox-5.2.12/src/VBox/Main/include/ovfreader.h VirtualBox-5.2.12.tpg/src/VBox/Main/include/ovfreader.h
--- VirtualBox-5.2.12/src/VBox/Main/include/ovfreader.h	2018-05-09 16:38:28.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Main/include/ovfreader.h	2018-06-15 12:44:53.610162147 +0000
@@ -153,8 +153,8 @@
     CIMOSType_CIMOS_NovellOES   = 86,
     CIMOSType_CIMOS_NovellLinuxDesktop  = 87,
     CIMOSType_CIMOS_SunJavaDesktopSystem    = 88,
-    CIMOSType_CIMOS_Mandriva    = 89,
-    CIMOSType_CIMOS_Mandriva_64 = 90,
+    CIMOSType_CIMOS_OpenMandriva    = 89,
+    CIMOSType_CIMOS_OpenMandriva_64 = 90,
     CIMOSType_CIMOS_TurboLinux  = 91,
     CIMOSType_CIMOS_TurboLinux_64   = 92,
     CIMOSType_CIMOS_Ubuntu  = 93,
diff -Naur VirtualBox-5.2.12/src/VBox/Main/src-all/Global.cpp VirtualBox-5.2.12.tpg/src/VBox/Main/src-all/Global.cpp
--- VirtualBox-5.2.12/src/VBox/Main/src-all/Global.cpp	2018-05-09 16:38:28.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Main/src-all/Global.cpp	2018-06-15 12:44:53.610162147 +0000
@@ -250,15 +250,15 @@
       1024,  16,  8 * _1G64, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
         StorageControllerType_IntelAhci, StorageBus_SATA, ChipsetType_PIIX3, AudioControllerType_AC97, AudioCodecType_AD1980  },
 
-    { "Linux",   "Linux",             "Mandriva",           "Mandriva (32-bit)",
-      VBOXOSTYPE_Mandriva,        VBOXOSHINT_RTCUTC | VBOXOSHINT_USBTABLET | VBOXOSHINT_X2APIC,
-      1024,  16,  8 * _1G64, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
+    { "Linux",   "Linux",             "OpenMandriva",           "OpenMandriva (32-bit)",
+      VBOXOSTYPE_OpenMandriva,        VBOXOSHINT_RTCUTC | VBOXOSHINT_USBTABLET | VBOXOSHINT_X2APIC,
+      1024,  16,  20 * _1G64, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
         StorageControllerType_IntelAhci, StorageBus_SATA, ChipsetType_PIIX3, AudioControllerType_AC97, AudioCodecType_AD1980  },
 
-    { "Linux",   "Linux",             "Mandriva_64",        "Mandriva (64-bit)",
-      VBOXOSTYPE_Mandriva_x64,    VBOXOSHINT_64BIT | VBOXOSHINT_HWVIRTEX | VBOXOSHINT_IOAPIC | VBOXOSHINT_RTCUTC
+    { "Linux",   "Linux",             "OpenMandriva_64",        "OpenMandriva (64-bit)",
+      VBOXOSTYPE_OpenMandriva_x64,    VBOXOSHINT_64BIT | VBOXOSHINT_HWVIRTEX | VBOXOSHINT_IOAPIC | VBOXOSHINT_RTCUTC
                                 | VBOXOSHINT_USBTABLET | VBOXOSHINT_X2APIC,
-      1024,  16,  8 * _1G64, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
+      1024,  16,  20 * _1G64, NetworkAdapterType_I82540EM, 0, StorageControllerType_PIIX4, StorageBus_IDE,
         StorageControllerType_IntelAhci, StorageBus_SATA, ChipsetType_PIIX3, AudioControllerType_AC97, AudioCodecType_AD1980  },
 
     { "Linux",   "Linux",             "RedHat",             "Red Hat (32-bit)",
diff -Naur VirtualBox-5.2.12/src/VBox/Main/src-server/ApplianceImpl.cpp VirtualBox-5.2.12.tpg/src/VBox/Main/src-server/ApplianceImpl.cpp
--- VirtualBox-5.2.12/src/VBox/Main/src-server/ApplianceImpl.cpp	2018-05-09 16:38:29.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Main/src-server/ApplianceImpl.cpp	2018-06-15 12:44:53.611162147 +0000
@@ -165,8 +165,8 @@
     { ovf::CIMOSType_CIMOS_SunJavaDesktopSystem,                 VBOXOSTYPE_Linux },
     { ovf::CIMOSType_CIMOS_TurboLinux,                           VBOXOSTYPE_Turbolinux },
     { ovf::CIMOSType_CIMOS_TurboLinux_64,                        VBOXOSTYPE_Turbolinux_x64 },
-    { ovf::CIMOSType_CIMOS_Mandriva,                             VBOXOSTYPE_Mandriva },
-    { ovf::CIMOSType_CIMOS_Mandriva_64,                          VBOXOSTYPE_Mandriva_x64 },
+    { ovf::CIMOSType_CIMOS_OpenMandriva,                         VBOXOSTYPE_OpenMandriva },
+    { ovf::CIMOSType_CIMOS_OpenMandriva_64,                      VBOXOSTYPE_OpenMandriva_x64 },
     { ovf::CIMOSType_CIMOS_Ubuntu,                               VBOXOSTYPE_Ubuntu },
     { ovf::CIMOSType_CIMOS_Ubuntu_64,                            VBOXOSTYPE_Ubuntu_x64 },
     { ovf::CIMOSType_CIMOS_Debian,                               VBOXOSTYPE_Debian },
@@ -228,7 +228,7 @@
     {"SUSE",          VBOXOSTYPE_OpenSUSE},
     {"Novell",        VBOXOSTYPE_OpenSUSE},
     {"Red Hat",       VBOXOSTYPE_RedHat},
-    {"Mandriva",      VBOXOSTYPE_Mandriva},
+    {"OpenMandriva",  VBOXOSTYPE_OpenMandriva},
     {"Ubuntu",        VBOXOSTYPE_Ubuntu},
     {"Debian",        VBOXOSTYPE_Debian},
     {"QNX",           VBOXOSTYPE_QNX},
@@ -257,7 +257,7 @@
     {"SUSE",          VBOXOSTYPE_OpenSUSE_x64},
     {"Novell",        VBOXOSTYPE_OpenSUSE_x64},
     {"Red Hat",       VBOXOSTYPE_RedHat_x64},
-    {"Mandriva",      VBOXOSTYPE_Mandriva_x64},
+    {"OpenMandriva",  VBOXOSTYPE_OpenMandriva_x64},
     {"Ubuntu",        VBOXOSTYPE_Ubuntu_x64},
     {"Debian",        VBOXOSTYPE_Debian_x64},
     {"Linux 2.4",     VBOXOSTYPE_Linux24_x64},
diff -Naur VirtualBox-5.2.12/src/VBox/Main/xml/Settings.cpp VirtualBox-5.2.12.tpg/src/VBox/Main/xml/Settings.cpp
--- VirtualBox-5.2.12/src/VBox/Main/xml/Settings.cpp	2018-05-09 16:38:31.000000000 +0000
+++ VirtualBox-5.2.12.tpg/src/VBox/Main/xml/Settings.cpp	2018-06-15 12:44:53.612162147 +0000
@@ -5095,7 +5095,7 @@
     { "opensuse", "OpenSUSE" },
     { "fedoracore", "Fedora" },
     { "gentoo", "Gentoo" },
-    { "mandriva", "Mandriva" },
+    { "openmandriva", "OpenMandriva" },
     { "redhat", "RedHat" },
     { "ubuntu", "Ubuntu" },
     { "xandros", "Xandros" },
