diff -up VirtualBox-5.2.10/src/libs/xpcom18a4/python/src/ErrorUtils.cpp.omv~ VirtualBox-5.2.10/src/libs/xpcom18a4/python/src/ErrorUtils.cpp
--- VirtualBox-5.2.10/src/libs/xpcom18a4/python/src/ErrorUtils.cpp.omv~	2018-04-27 01:37:19.442871087 +0200
+++ VirtualBox-5.2.10/src/libs/xpcom18a4/python/src/ErrorUtils.cpp	2018-04-27 01:37:33.612523461 +0200
@@ -440,7 +440,7 @@ char *PyTraceback_AsString(PyObject *exc
 #if PY_MAJOR_VERSION <= 2
 	char *tempResult = PyString_AsString(obResult);
 #else
-	char *tempResult = PyUnicode_AsUTF8(obResult);
+	const char *tempResult = PyUnicode_AsUTF8(obResult);
 #endif
 	result = (char *)PyMem_Malloc(strlen(tempResult)+1);
 	if (result==NULL)
diff -up VirtualBox-5.2.10/src/libs/xpcom18a4/python/src/PyXPCOM.h.omv~ VirtualBox-5.2.10/src/libs/xpcom18a4/python/src/PyXPCOM.h
