diff -up VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp.0016~ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp
--- VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp.0016~	2018-12-19 00:10:07.476926771 +0100
+++ VirtualBox-6.0.0/src/VBox/Frontends/VirtualBox/src/widgets/UINameAndSystemEditor.cpp	2018-12-19 00:11:34.158542152 +0100
@@ -251,15 +251,15 @@ void UINameAndSystemEditor::sltFamilyCha
         if (iIndexWin7 != -1)
             m_pComboType->setCurrentIndex(iIndexWin7);
     }
-    /* Or select Oracle Linux item for Linux family as default: */
+    /* Or select Mandriva Linux item for Linux family as default: */
     else if (m_strFamilyId == "Linux")
     {
-        QString strDefaultID = "Oracle";
+        QString strDefaultID = "Mandriva";
         if (ARCH_BITS == 64 && m_fSupportsHWVirtEx && m_fSupportsLongMode)
             strDefaultID += "_64";
-        const int iIndexUbuntu = m_pComboType->findData(strDefaultID, TypeID);
-        if (iIndexUbuntu != -1)
-            m_pComboType->setCurrentIndex(iIndexUbuntu);
+        const int iIndexMandriva = m_pComboType->findData(strDefaultID, TypeID);
+        if (iIndexMandriva != -1)
+            m_pComboType->setCurrentIndex(iIndexMandriva);
     }
     /* Else simply select the first one present: */
     else
