diff -Naur VirtualBox-5.1.0/src/VBox/Installer/linux/VBox.sh VirtualBox-5.1.0.tpg/src/VBox/Installer/linux/VBox.sh
--- VirtualBox-5.1.0/src/VBox/Installer/linux/VBox.sh	2016-07-12 14:54:27.000000000 +0000
+++ VirtualBox-5.1.0.tpg/src/VBox/Installer/linux/VBox.sh	2016-07-22 20:05:15.034670870 +0000
@@ -62,7 +62,7 @@
          available for the current kernel (`uname -r`) or it failed to
          load. Please recompile the kernel module and install it by
 
-           sudo /sbin/vboxconfig
+           sudo dkms autoinstall --verbose
 
          You will not be able to start VMs until this problem is fixed.
 EOF
@@ -71,6 +71,8 @@
 WARNING: The character device /dev/vboxdrv does not exist. Try
 
            sudo /sbin/vboxconfig
+           or
+           sudo dkms autoinstall --verbose
 
          and if that is not successful, try to re-install the package.
 
--- VirtualBox-5.2.28/src/VBox/VMM/VMMR3/VM.cpp.0001~	2019-05-01 13:22:09.436593592 +0100
+++ VirtualBox-5.2.28/src/VBox/VMM/VMMR3/VM.cpp	2019-05-01 13:26:29.309729489 +0100
@@ -278,7 +278,7 @@
                                   "Secure Boot that the modules are signed if necessary in the right "
                                   "way for your host system.  Then try to recompile and reload the "
                                   "kernel modules by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dkms autoinstall --verbose' as root");
                     break;
 #endif
 
@@ -346,7 +346,7 @@
                                   "or you are using EFI Secure Boot and the module is not signed "
                                   "in the right way for your system.  If necessary, try setting up "
                                   "the kernel module again by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dkms autoinstall --verbose' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not loaded");
 #endif
@@ -390,7 +390,7 @@
                                   "or you are using EFI Secure Boot and the module is not signed "
                                   "in the right way for your system.  If necessary, try setting up "
                                   "the kernel module again by executing "
-                                  "'/sbin/vboxconfig' as root");
+                                  "'dkms autoinstall --verbose' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not installed");
 #endif
