diff -up VirtualBox-7.2.0/configure.9~ VirtualBox-7.2.0/configure
--- VirtualBox-7.2.0/configure.9~	2025-08-13 21:37:05.000000000 +0200
+++ VirtualBox-7.2.0/configure	2025-08-18 02:26:38.405106403 +0200
@@ -3016,7 +3016,7 @@ if [ "$OS" = "linux" ]; then
   fi
   check_compiler_h
   # tools/common/makeself*
-  [ $OSE -ge 1 ] && check_makeself
+  #[ $OSE -ge 1 ] && check_makeself
 fi
 
 [ -n "$SETUP_WINE" ] && setup_wine
