--- VirtualBox-5.0.0_BETA3/src/VBox/Additions/common/VBoxGuestLib/Makefile.kmk	2015-05-07 18:32:02.000000000 +0000
+++ VirtualBox-5.0.0_BETA3/src/VBox/Additions/common/VBoxGuestLib/Makefile.kmk.tpg	2015-05-09 11:52:10.975858850 +0000
@@ -39,10 +39,12 @@
 	VBoxGuestR3Lib \
 	VBoxGuestR3LibShared
 ifndef VBOX_ONLY_VALIDATIONKIT
+ifndef VBOX_USE_SYSTEM_XORG_HEADERS
  if1of ($(KBUILD_TARGET), freebsd linux netbsd openbsd)
   LIBRARIES += \
   	VBoxGuestR3LibXFree86
  endif
+endif
  if1of ($(KBUILD_TARGET), freebsd linux netbsd openbsd solaris)
   LIBRARIES += \
   	VBoxGuestR3LibXOrg
