--- VirtualBox-5.2.0/src/VBox/Additions/linux/installer/vboxadd-service.sh.0004~	2017-10-31 11:49:42.519439224 +0100
+++ VirtualBox-5.2.0/src/VBox/Additions/linux/installer/vboxadd-service.sh	2017-10-31 11:51:47.959301310 +0100
@@ -21,8 +21,10 @@
 #
 ### BEGIN INIT INFO
 # Provides:       vboxadd-service
-# Required-Start: vboxadd
-# Required-Stop:  vboxadd
+# Should-Start:   dkms
+# Required-Start:
+# Required-Stop:
+# Short-Description: VirtualBox Additions Service
 # Default-Start:  2 3 4 5
 # Default-Stop:   0 1 6
 # X-Conflicts-With: systemd-timesyncd.service
