diff -Naur VirtualBox-6.0.14/Config.kmk VirtualBox-6.0.14.tpg/Config.kmk
--- VirtualBox-6.0.14/Config.kmk	2019-10-10 18:25:35.000000000 +0000
+++ VirtualBox-6.0.14.tpg/Config.kmk	2019-10-31 15:08:08.235983271 +0000
@@ -4069,23 +4069,27 @@
 # SDKs for external libraries.
 #
 SDK_VBOX_LIBXML2           = .
-SDK_VBOX_LIBXML2_INCS     ?= $(PATH_ROOT)/src/libs/libxml2-2.9.4/include
+SDK_VBOX_LIBXML2_INCS     ?= $(pkg-config libxml-2.0 --variable=includedir)
 SDK_VBOX_LIBXML2_DEFS     ?= _REENTRANT LIBXML_STATIC LIBXML_STATIC_FOR_DLL
 SDK_VBOX_LIBXML2_DEFS.win ?= WIN32 _WINDOWS _MBCS HAVE_WIN32_THREADS HAVE_COMPILER_TLS
 # Note: no linking to LIB here, we do that explicitly in src/VBox/Runtime/Makefile.kmk to link
 #       libxml against VBoxRT
 
 SDK_VBOX_LIBPNG       = .
-SDK_VBOX_LIBPNG_INCS ?= $(PATH_ROOT)/src/libs/libpng-1.6.36
-SDK_VBOX_LIBPNG_LIBS ?= $(PATH_STAGE_LIB)/VBox-libpng$(VBOX_SUFF_LIB)
+SDK_VBOX_LIBPNG_INCS ?= $(pkg-config libpng --variable=includedir)
+SDK_VBOX_LIBPNG_LIBS ?= $(pkg-config libpng --libs)
+
+SDK_VBOX_LIBLZF       = .
+SDK_VBOX_LIBLZF_INCS ?= $(pkg-config liblzf --variable=includedir)
+SDK_VBOX_LIBLZF_LIBS ?= $(pkg-config liblzf --libs)
 
 SDK_VBOX_ZLIB         = .
 if1of ($(KBUILD_TARGET), os2 solaris win)
  SDK_VBOX_ZLIB_INCS  ?= $(PATH_ROOT)/src/libs/zlib-1.2.11
  SDK_VBOX_ZLIB_LIBS  ?= $(PATH_STAGE_LIB)/VBox-zlib$(VBOX_SUFF_LIB)
 else
- SDK_VBOX_ZLIB_INCS  ?=
- SDK_VBOX_ZLIB_LIBS  ?= z
+ SDK_VBOX_ZLIB_INCS  ?= $(pkg-config zlib --variable=includedir)
+ SDK_VBOX_ZLIB_LIBS  ?= $(pkg-config zlib --libs)
  SDK_VBOX_ZLIB_DEFS.linux ?= _LARGEFILE64_SOURCE
 endif
 
@@ -4134,28 +4138,28 @@
 
 ifdef VBOX_WITH_LIBCURL
 SDK_VBOX_LIBCURL       = .
-SDK_VBOX_LIBCURL_INCS ?= $(PATH_ROOT)/src/libs/curl-7.64.0/include
-SDK_VBOX_LIBCURL_LIBS ?= $(PATH_STAGE_LIB)/VBox-libcurl$(VBOX_SUFF_LIB)
+SDK_VBOX_LIBCURL_INCS ?= $(pkg-config libcurl --variable=includedir)
+SDK_VBOX_LIBCURL_LIBS ?= $(pkg-config libcurl --libs)
 SDK_VBOX_LIBCURL_DEFS ?= BUILDING_LIBCURL
 
 SDK_VBOX_LIBCURL-x86           = .
-SDK_VBOX_LIBCURL-x86_INCS     ?= $(PATH_ROOT)/src/libs/curl-7.64.0/include
-SDK_VBOX_LIBCURL-x86_LIBS.x86 ?= $(PATH_STAGE_LIB)/VBox-libcurl-x86$(VBOX_SUFF_LIB)
+SDK_VBOX_LIBCURL-x86_INCS     ?= $(pkg-config libcurl --variable=includedir)
+SDK_VBOX_LIBCURL-x86_LIBS.x86 ?= $(pkg-config libcurl --libs)
 SDK_VBOX_LIBCURL-x86_DEFS     ?= BUILDING_LIBCURL
 endif
 
 ifdef VBOX_WITH_LIBVPX
  SDK_VBOX_VPX       = .
  SDK_VBOX_VPX_DEFAULT_INCS := $(PATH_ROOT)/src/libs/libvpx-1.6.1
- SDK_VBOX_VPX_INCS ?= $(SDK_VBOX_VPX_DEFAULT_INCS)
- SDK_VBOX_VPX_LIBS ?= $(PATH_STAGE_LIB)/VBox-libvpx$(VBOX_SUFF_LIB)
+ SDK_VBOX_VPX_INCS ?= $(pkg-config vpx --variable=includedir)
+ SDK_VBOX_VPX_LIBS ?= $(pkg-config vpx --libs)
 endif
 
 ifdef VBOX_WITH_LIBOPUS
  SDK_VBOX_OPUS       = .
- SDK_VBOX_OPUS_DEFAULT_INCS := $(PATH_ROOT)/src/libs/libopus-1.2.1/include
- SDK_VBOX_OPUS_INCS ?= $(SDK_VBOX_OPUS_DEFAULT_INCS)
- SDK_VBOX_OPUS_LIBS ?= $(PATH_STAGE_LIB)/VBox-libopus$(VBOX_SUFF_LIB)
+ SDK_VBOX_OPUS_DEFAULT_INCS := $(PATH_ROOT)/src/libs/libvpx-1.2.1/include
+ SDK_VBOX_OPUS_INCS ?= $(pkg-config opus --variable=includedir)
+ SDK_VBOX_OPUS_LIBS ?= $(pkg-config opus --libs)
 endif
 
 #
@@ -5398,7 +5402,7 @@
  	$(PATH_STAGE_LIB)/VBox-zlib-static$(VBOX_SUFF_LIB)
 else
  TEMPLATE_VBoxR3Static_LIBS        += \
- 	$(PATH_STAGE_LIB)/VBox-liblzf$(VBOX_SUFF_LIB) \
+ 	$(SDK_VBOX_LIBLZF_LIBS) \
  	$(SDK_VBOX_ZLIB_LIBS)
  if1of ($(KBUILD_TARGET),darwin freebsd)
   TEMPLATE_VBoxR3Static_LIBS       += iconv
diff -Naur VirtualBox-6.0.14/src/VBox/Runtime/Makefile.kmk VirtualBox-6.0.14.tpg/src/VBox/Runtime/Makefile.kmk
--- VirtualBox-6.0.14/src/VBox/Runtime/Makefile.kmk	2019-10-10 18:35:56.000000000 +0000
+++ VirtualBox-6.0.14.tpg/src/VBox/Runtime/Makefile.kmk	2019-10-31 15:08:08.237983271 +0000
@@ -292,7 +292,7 @@
 	include \
 	$(SDK_VBOX_ZLIB_INCS)
 RuntimeR3_INCS         += \
-	$(PATH_ROOT)/src/libs/liblzf-3.4
+	$(SDK_VBOX_LIBLZF_INCS)
 
 # RuntimeR3_INCS.solaris  = \
 #    /usr/include
@@ -2167,7 +2167,7 @@
 VBoxRT_INCS.$(KBUILD_TARGET)  := $(RuntimeR3_INCS.$(KBUILD_TARGET))
 VBoxRT_INCS.$(KBUILD_TARGET).$(KBUILD_TARGET_ARCH) := $(RuntimeR3_INCS.$(KBUILD_TARGET).$(KBUILD_TARGET_ARCH))
 VBoxRT_LIBS                    = \
-	$(PATH_STAGE_LIB)/VBox-liblzf$(VBOX_SUFF_LIB)
+	$(SDK_VBOX_LIBLZF_LIBS)
 if1of ($(KBUILD_TARGET_ARCH), amd64 x86)
 VBoxRT_LIBS                   += \
 	$(PATH_STAGE_LIB)/SUPR3$(VBOX_SUFF_LIB)
@@ -2314,7 +2314,7 @@
 VBoxRT-x86_SDKS  += VBOX_OPENSSL-x86
 
 VBoxRT-x86_LIBS = \
-	$(PATH_STAGE_LIB)/VBox-liblzf-x86$(VBOX_SUFF_LIB)
+	$(SDK_VBOX_LIBLZF_LIBS)
 if1of ($(KBUILD_TARGET_ARCH), amd64 x86)
 VBoxRT-x86_LIBS += \
 	$(PATH_STAGE_LIB)/SUPR3-x86$(VBOX_SUFF_LIB)
diff -Naur VirtualBox-6.0.14/src/VBox/Storage/testcase/Makefile.kmk VirtualBox-6.0.14.tpg/src/VBox/Storage/testcase/Makefile.kmk
--- VirtualBox-6.0.14/src/VBox/Storage/testcase/Makefile.kmk	2019-10-10 18:36:29.000000000 +0000
+++ VirtualBox-6.0.14.tpg/src/VBox/Storage/testcase/Makefile.kmk	2019-10-31 15:11:22.649969762 +0000
@@ -137,7 +137,7 @@
 	$(PATH_STAGE_LIB)/VBox-zlib-static$(VBOX_SUFF_LIB)
  else
   vbox-img_LIBS += \
-	$(PATH_STAGE_LIB)/VBox-liblzf$(VBOX_SUFF_LIB) \
+	$(SDK_VBOX_LIBLZF_LIBS) \
 	$(SDK_VBOX_ZLIB_LIBS)
  endif
  ifeq ($(KBUILD_TARGET),linux)
@@ -183,7 +183,7 @@
 	$(PATH_STAGE_LIB)/VBox-zlib-static$(VBOX_SUFF_LIB)
  else
   vdkeystoremgr_LIBS += \
-	$(PATH_STAGE_LIB)/VBox-liblzf$(VBOX_SUFF_LIB) \
+	$(SDK_VBOX_LIBLZF_LIBS) \
 	$(SDK_VBOX_ZLIB_LIBS)
  endif
  ifeq ($(KBUILD_TARGET),linux)
diff -Naur VirtualBox-6.0.14/src/libs/Makefile.kmk VirtualBox-6.0.14.tpg/src/libs/Makefile.kmk
--- VirtualBox-6.0.14/src/libs/Makefile.kmk	2019-10-10 18:36:58.000000000 +0000
+++ VirtualBox-6.0.14.tpg/src/libs/Makefile.kmk	2019-10-31 15:08:08.237983271 +0000
@@ -25,7 +25,7 @@
 #
 
 # Compression libs used by IPRT and <what was it again?>.
-include $(PATH_SUB_CURRENT)/liblzf-3.4/Makefile.kmk
+#include $(PATH_SUB_CURRENT)/liblzf-3.4/Makefile.kmk
 if  !defined(VBOX_ONLY_SDK) \
  && "$(SDK_VBOX_ZLIB_INCS)" == "$(PATH_ROOT)/src/libs/zlib-1.2.11"
  include $(PATH_SUB_CURRENT)/zlib-1.2.11/Makefile.kmk
