--- VirtualBox-5.1.26/src/VBox/ExtPacks/VBoxDTrace/include/VBoxDTraceTypes.h.omv~	2017-07-31 09:48:04.135456028 +0200
+++ VirtualBox-5.1.26/src/VBox/ExtPacks/VBoxDTrace/include/VBoxDTraceTypes.h	2017-07-31 10:17:30.333011149 +0200
@@ -56,7 +56,9 @@ typedef uint32_t                    id_t
 typedef uint32_t                    zoneid_t;
 #endif
 #if !defined(NGREG) || !defined(RT_OS_LINUX)
+#ifndef _SYS_UCONTEXT_H // sys/context.h defines greg_t
 typedef RTCCINTREG                  greg_t;
+#endif
 #else
 AssertCompileSize(greg_t, sizeof(RTCCINTREG));
 #endif
